{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Multi-Json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "import csv\n",
    "import os\n",
    "import argparse"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "results_dir = \"./results\"\n",
    "\n",
    "# Reads all of the json files in the dir\n",
    "def list_json_files(directory):\n",
    "    json_files = []\n",
    "    for root, dirs, files in os.walk(directory):\n",
    "        for file in files:\n",
    "            if file.endswith('.json'):\n",
    "                json_files.append(os.path.join(root, file))\n",
    "    return json_files\n",
    "\n",
    "list_dir = list_json_files(results_dir)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "def above_two(data_records):\n",
    "    index=[]\n",
    "    for i in range(len(data_records)):\n",
    "        modules = data_records[i][\"modules\"].keys()\n",
    "        if len(modules) > 2:\n",
    "            # print(i, len(modules))\n",
    "            index.append(i)\n",
    "        else:\n",
    "            continue\n",
    "    return index\n",
    "\n",
    "def process_files(file_paths):\n",
    "    \n",
    "    csv_filename = 'comb_output.csv'\n",
    "    \n",
    "    # Check if the output file already exists\n",
    "    file_exists = os.path.isfile(csv_filename)\n",
    "    \n",
    "    with open(csv_filename, 'a', newline='') as csvfile:\n",
    "        writer = csv.writer(csvfile)\n",
    "        \n",
    "        if not file_exists:\n",
    "            writer.writerow([\"Bin_ID\", \"Contig_ID\", \"Gene_Coordinates\", \"Ctg_Number\", \"Ctg_Tag_Id\", \"Mibig_biosynthetic_gene_cluster\", \"Mibig_Product\", \"Blast_name\", \"Blast_genecluster\",\"Blast_annotation\",\"Blast_perc_coverage\", \"Blast_perc_ident\", \"Blast_blastscore\", \"Blast_evalue\", \"Blast_locus_tag\" ])\n",
    "        \n",
    "        for file_path in file_paths:\n",
    "\n",
    "            with open(file_path, 'r') as files:\n",
    "                data = json.load(files)\n",
    "                index_ = above_two(data[\"records\"])\n",
    "                \n",
    "                for i in index_:\n",
    "                    bin_id = data[\"input_file\"]\n",
    "                    contig_id = data[\"records\"][i][\"modules\"][\"antismash.modules.clusterblast\"][\"record_id\"]\n",
    "                    blast_info = data[\"records\"][i][\"modules\"][\"antismash.modules.clusterblast\"][\"general\"][\"results\"][0][\"ranking\"][1][1][\"pairings\"]\n",
    "                    mibig_entries = data[\"records\"][i][\"modules\"][\"antismash.modules.clusterblast\"][\"knowncluster\"][\"mibig_entries\"][\"1\"]\n",
    "\n",
    "                    # Dict for each ctg_tagid\n",
    "                    blast_ = {}\n",
    "                    for blast in blast_info:\n",
    "                        ctg_name = blast[0].split(\"|\")[4]\n",
    "                        blast_[\"ctg_cordinates\"] = blast[0].split(\"|\")[2]\n",
    "                        blast_name, blast_genecluster, blast_start, blast_end, blast_strand, blast_annotation, blast_perc_ident, blast_blastscore, blast_perc_coverage, blast_evalue, blast_locus_tag  = blast[2].values()\n",
    "                        blast_[ctg_name] = {\"blast_name\" : blast_name, \n",
    "                                            \"blast_genecluster\": blast_genecluster,\n",
    "                                            \"blast_annotation\": blast_annotation,\n",
    "                                            \"blast_perc_coverage\":blast_perc_coverage,\n",
    "                                            \"blast_perc_ident\": blast_perc_ident,\n",
    "                                            \"blast_blastscore\": blast_blastscore,\n",
    "                                            \"blast_evalue\": blast_evalue,\n",
    "                                            \"blast_locus_tag\": blast_locus_tag,\n",
    "                                            \"cordinates\": blast[0].split(\"|\")[2] }\n",
    "\n",
    "                    blast_keys = list(blast_.keys())[1:]\n",
    "                    mibig_entries_keys = [ ctg_tagid for ctg_tagid in mibig_entries.keys()]\n",
    "        \n",
    "\n",
    "                    for ix, vl in  enumerate(blast_keys):\n",
    "                        ctg_id = vl.split(\"_\")[0]\n",
    "                        ctg_tagid = vl\n",
    "                        ctg_cordinates = blast_[vl][\"cordinates\"]\n",
    "                        blast_name = blast_[vl][\"blast_name\"]\n",
    "                        blast_genecluster= blast_[vl][\"blast_genecluster\"]\n",
    "                        blast_annotation = blast_[vl][\"blast_annotation\"]\n",
    "                        blast_perc_coverage = blast_[vl][\"blast_perc_coverage\"]\n",
    "                        blast_perc_ident = blast_[vl][\"blast_perc_ident\"]\n",
    "                        blast_blastscore = blast_[vl][\"blast_blastscore\"]\n",
    "                        blast_evalue = blast_[vl][\"blast_evalue\"]\n",
    "                        blast_locus_tag = blast_[vl][\"blast_locus_tag\"]\n",
    "                        biosynthetic_gene_cluster_id=None \n",
    "                        mibig_entries_product=None\n",
    "\n",
    "                        if vl not in mibig_entries_keys:\n",
    "                            writer.writerow([\n",
    "                                    bin_id, \n",
    "                                    contig_id,\n",
    "                                    ctg_cordinates, \n",
    "                                    ctg_id, \n",
    "                                    ctg_tagid, \n",
    "                                    biosynthetic_gene_cluster_id, \n",
    "                                    mibig_entries_product, \n",
    "                                    blast_name,\n",
    "                                    blast_genecluster,\n",
    "                                    blast_annotation,\n",
    "                                    blast_perc_coverage,\n",
    "                                    blast_perc_ident,\n",
    "                                    blast_blastscore,\n",
    "                                    blast_evalue,\n",
    "                                    blast_locus_tag\n",
    "                                ])\n",
    "\n",
    "\n",
    "\n",
    "                    for ctg_tagid, vl in mibig_entries.items():\n",
    "                        ctg_id = ctg_tagid.split(\"_\")[0]\n",
    "                        mibig_entry = vl[0]\n",
    "                        biosynthetic_gene_cluster_id = mibig_entry[2]\n",
    "                        mibig_entries_product = mibig_entry[4]\n",
    "                        blast_info_ = blast_.get(ctg_tagid)\n",
    "\n",
    "                        if blast_info_:\n",
    "                            ctg_cordinates = blast_info_[\"cordinates\"]\n",
    "                            blast_name = blast_info_[\"blast_name\"]\n",
    "                            blast_genecluster= blast_info_[\"blast_genecluster\"]\n",
    "                            blast_annotation = blast_info_[\"blast_annotation\"]\n",
    "                            blast_perc_coverage = blast_info_[\"blast_perc_coverage\"]\n",
    "                            blast_perc_ident = blast_info_[\"blast_perc_ident\"]\n",
    "                            blast_blastscore = blast_info_[\"blast_blastscore\"]\n",
    "                            blast_evalue = blast_info_[\"blast_evalue\"]\n",
    "                            blast_locus_tag = blast_info_[\"blast_locus_tag\"]\n",
    "\n",
    "                        else:\n",
    "                            ctg_cordinates = None\n",
    "                            blast_name = None\n",
    "                            blast_genecluster= None\n",
    "                            blast_annotation = None\n",
    "                            blast_perc_coverage = None\n",
    "                            blast_perc_ident = None\n",
    "                            blast_blastscore = None\n",
    "                            blast_evalue = None\n",
    "                            blast_locus_tag = None\n",
    "                            \n",
    "                        writer.writerow([\n",
    "                            bin_id, \n",
    "                            contig_id,\n",
    "                            ctg_cordinates, \n",
    "                            ctg_id, \n",
    "                            ctg_tagid, \n",
    "                            biosynthetic_gene_cluster_id, \n",
    "                            mibig_entries_product, \n",
    "                            blast_name,\n",
    "                            blast_genecluster,\n",
    "                            blast_annotation,\n",
    "                            blast_perc_coverage,\n",
    "                            blast_perc_ident,\n",
    "                            blast_blastscore,\n",
    "                            blast_evalue,\n",
    "                            blast_locus_tag\n",
    "                        ])\n",
    "            \n",
    "\n",
    "process_files(list_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Bin_ID</th>\n",
       "      <th>Contig_ID</th>\n",
       "      <th>Gene_Coordinates</th>\n",
       "      <th>Ctg_Number</th>\n",
       "      <th>Ctg_Tag_Id</th>\n",
       "      <th>Mibig_biosynthetic_gene_cluster</th>\n",
       "      <th>Mibig_Product</th>\n",
       "      <th>Blast_name</th>\n",
       "      <th>Blast_genecluster</th>\n",
       "      <th>Blast_annotation</th>\n",
       "      <th>Blast_perc_coverage</th>\n",
       "      <th>Blast_perc_ident</th>\n",
       "      <th>Blast_blastscore</th>\n",
       "      <th>Blast_evalue</th>\n",
       "      <th>Blast_locus_tag</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>29585-29864</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_29</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CG392_RS04935</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>hypothetical_protein</td>\n",
       "      <td>98.913043</td>\n",
       "      <td>100.0</td>\n",
       "      <td>186.0</td>\n",
       "      <td>2.970000e-61</td>\n",
       "      <td>WP_116438264.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>29958-30246</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_30</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CG392_RS04940</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>type_II_toxin-antitoxin_system_RelB/DinJ_famil...</td>\n",
       "      <td>100.000000</td>\n",
       "      <td>100.0</td>\n",
       "      <td>192.0</td>\n",
       "      <td>1.900000e-63</td>\n",
       "      <td>WP_116438265.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>30616-31054</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_31</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CG392_RS04945</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>ORF6N_domain-containing_protein</td>\n",
       "      <td>79.310345</td>\n",
       "      <td>74.0</td>\n",
       "      <td>160.0</td>\n",
       "      <td>5.670000e-50</td>\n",
       "      <td>CG392_RS04945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>31058-32072</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_32</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CG392_RS04950</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>Abi_family_protein</td>\n",
       "      <td>100.000000</td>\n",
       "      <td>100.0</td>\n",
       "      <td>679.0</td>\n",
       "      <td>2.570000e-248</td>\n",
       "      <td>WP_116438267.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>33418-38701</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_34</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CG392_RS07955</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>hypothetical_protein</td>\n",
       "      <td>45.397727</td>\n",
       "      <td>45.0</td>\n",
       "      <td>571.0</td>\n",
       "      <td>6.240000e-181</td>\n",
       "      <td>CG392_RS07955</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>MG_323757.bin1.fa</td>\n",
       "      <td>NODE_83_length_8923_cov_7.096640</td>\n",
       "      <td>813-996</td>\n",
       "      <td>ctg59</td>\n",
       "      <td>ctg59_2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>KW861_RS14645</td>\n",
       "      <td>NZ_CP078505_c3139533-3154876</td>\n",
       "      <td>Blp_family_class_II_bacteriocin</td>\n",
       "      <td>100.000000</td>\n",
       "      <td>83.0</td>\n",
       "      <td>80.0</td>\n",
       "      <td>5.870000e-20</td>\n",
       "      <td>WP_103301327.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>MG_323757.bin1.fa</td>\n",
       "      <td>NODE_83_length_8923_cov_7.096640</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ctg59</td>\n",
       "      <td>ctg59_6</td>\n",
       "      <td>BGC0000100</td>\n",
       "      <td>Polyketide</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>MG_323757.bin1.fa</td>\n",
       "      <td>NODE_110_length_7016_cov_6.405832</td>\n",
       "      <td>520-1279</td>\n",
       "      <td>ctg82</td>\n",
       "      <td>ctg82_1</td>\n",
       "      <td>BGC0002667</td>\n",
       "      <td>RiPP</td>\n",
       "      <td>A5816_RS11485</td>\n",
       "      <td>NZ_NGLI01000002_c555810-572363</td>\n",
       "      <td>LytTR_family_DNA-binding_domain-containing_pro...</td>\n",
       "      <td>98.809524</td>\n",
       "      <td>93.0</td>\n",
       "      <td>460.0</td>\n",
       "      <td>1.940000e-164</td>\n",
       "      <td>WP_010733065.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>MG_323757.bin1.fa</td>\n",
       "      <td>NODE_110_length_7016_cov_6.405832</td>\n",
       "      <td>1280-2594</td>\n",
       "      <td>ctg82</td>\n",
       "      <td>ctg82_2</td>\n",
       "      <td>BGC0000624</td>\n",
       "      <td>RiPP</td>\n",
       "      <td>A5816_RS11480</td>\n",
       "      <td>NZ_NGLI01000002_c555810-572363</td>\n",
       "      <td>GHKL_domain-containing_protein</td>\n",
       "      <td>72.311213</td>\n",
       "      <td>88.0</td>\n",
       "      <td>527.0</td>\n",
       "      <td>6.040000e-187</td>\n",
       "      <td>WP_080114483.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>MG_323757.bin1.fa</td>\n",
       "      <td>NODE_110_length_7016_cov_6.405832</td>\n",
       "      <td>2682-4854</td>\n",
       "      <td>ctg82</td>\n",
       "      <td>ctg82_3</td>\n",
       "      <td>BGC0002579</td>\n",
       "      <td>RiPP</td>\n",
       "      <td>A5816_RS11475</td>\n",
       "      <td>NZ_NGLI01000002_c555810-572363</td>\n",
       "      <td>peptide_cleavage/export_ABC_transporter</td>\n",
       "      <td>100.000000</td>\n",
       "      <td>91.0</td>\n",
       "      <td>1263.0</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>WP_010733063.1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>94 rows Ã— 15 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "               Bin_ID                           Contig_ID Gene_Coordinates  \\\n",
       "0   MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      29585-29864   \n",
       "1   MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      29958-30246   \n",
       "2   MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      30616-31054   \n",
       "3   MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      31058-32072   \n",
       "4   MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      33418-38701   \n",
       "..                ...                                 ...              ...   \n",
       "89  MG_323757.bin1.fa    NODE_83_length_8923_cov_7.096640          813-996   \n",
       "90  MG_323757.bin1.fa    NODE_83_length_8923_cov_7.096640              NaN   \n",
       "91  MG_323757.bin1.fa   NODE_110_length_7016_cov_6.405832         520-1279   \n",
       "92  MG_323757.bin1.fa   NODE_110_length_7016_cov_6.405832        1280-2594   \n",
       "93  MG_323757.bin1.fa   NODE_110_length_7016_cov_6.405832        2682-4854   \n",
       "\n",
       "   Ctg_Number Ctg_Tag_Id Mibig_biosynthetic_gene_cluster Mibig_Product  \\\n",
       "0        ctg3    ctg3_29                             NaN           NaN   \n",
       "1        ctg3    ctg3_30                             NaN           NaN   \n",
       "2        ctg3    ctg3_31                             NaN           NaN   \n",
       "3        ctg3    ctg3_32                             NaN           NaN   \n",
       "4        ctg3    ctg3_34                             NaN           NaN   \n",
       "..        ...        ...                             ...           ...   \n",
       "89      ctg59    ctg59_2                             NaN           NaN   \n",
       "90      ctg59    ctg59_6                      BGC0000100    Polyketide   \n",
       "91      ctg82    ctg82_1                      BGC0002667          RiPP   \n",
       "92      ctg82    ctg82_2                      BGC0000624          RiPP   \n",
       "93      ctg82    ctg82_3                      BGC0002579          RiPP   \n",
       "\n",
       "       Blast_name               Blast_genecluster  \\\n",
       "0   CG392_RS04935   NZ_NNRS01000004_c78906-120244   \n",
       "1   CG392_RS04940   NZ_NNRS01000004_c78906-120244   \n",
       "2   CG392_RS04945   NZ_NNRS01000004_c78906-120244   \n",
       "3   CG392_RS04950   NZ_NNRS01000004_c78906-120244   \n",
       "4   CG392_RS07955   NZ_NNRS01000004_c78906-120244   \n",
       "..            ...                             ...   \n",
       "89  KW861_RS14645    NZ_CP078505_c3139533-3154876   \n",
       "90            NaN                             NaN   \n",
       "91  A5816_RS11485  NZ_NGLI01000002_c555810-572363   \n",
       "92  A5816_RS11480  NZ_NGLI01000002_c555810-572363   \n",
       "93  A5816_RS11475  NZ_NGLI01000002_c555810-572363   \n",
       "\n",
       "                                     Blast_annotation  Blast_perc_coverage  \\\n",
       "0                                hypothetical_protein            98.913043   \n",
       "1   type_II_toxin-antitoxin_system_RelB/DinJ_famil...           100.000000   \n",
       "2                     ORF6N_domain-containing_protein            79.310345   \n",
       "3                                  Abi_family_protein           100.000000   \n",
       "4                                hypothetical_protein            45.397727   \n",
       "..                                                ...                  ...   \n",
       "89                    Blp_family_class_II_bacteriocin           100.000000   \n",
       "90                                                NaN                  NaN   \n",
       "91  LytTR_family_DNA-binding_domain-containing_pro...            98.809524   \n",
       "92                     GHKL_domain-containing_protein            72.311213   \n",
       "93            peptide_cleavage/export_ABC_transporter           100.000000   \n",
       "\n",
       "    Blast_perc_ident  Blast_blastscore   Blast_evalue Blast_locus_tag  \n",
       "0              100.0             186.0   2.970000e-61  WP_116438264.1  \n",
       "1              100.0             192.0   1.900000e-63  WP_116438265.1  \n",
       "2               74.0             160.0   5.670000e-50   CG392_RS04945  \n",
       "3              100.0             679.0  2.570000e-248  WP_116438267.1  \n",
       "4               45.0             571.0  6.240000e-181   CG392_RS07955  \n",
       "..               ...               ...            ...             ...  \n",
       "89              83.0              80.0   5.870000e-20  WP_103301327.1  \n",
       "90               NaN               NaN            NaN             NaN  \n",
       "91              93.0             460.0  1.940000e-164  WP_010733065.1  \n",
       "92              88.0             527.0  6.040000e-187  WP_080114483.1  \n",
       "93              91.0            1263.0   0.000000e+00  WP_010733063.1  \n",
       "\n",
       "[94 rows x 15 columns]"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "dataframe = pd.read_csv(\"./comb_output.csv\")\n",
    "dataframe.head(100)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['RiPP:Microcin', 'RiPP:Thiopeptide', 'Other', 'Alkaloid', 'NRP',\n",
       "       'NRP+Polyketide:Modular type I polyketide', 'RiPP:Lanthipeptide',\n",
       "       'NRP+Polyketide', 'Polyketide', 'Saccharide',\n",
       "       'Terpene+Polyketide:Type III polyketide', 'RiPP'], dtype=object)"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataframe[\"Mibig_Product\"].unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "# from sklearn.preprocessing import LabelEncoder\n",
    "# dataframe_copy = dataframe.copy()\n",
    "\n",
    "# # Initialize LabelEncoder\n",
    "# label_encoder = LabelEncoder()\n",
    "\n",
    "# # Encode the 'Category' column\n",
    "# dataframe_copy['Mibig_Product_Encoded'] = label_encoder.fit_transform(dataframe_copy['Mibig_Product'])\n",
    "\n",
    "# # Print the encoding mappings\n",
    "# print(\"Encoding mappings:\")\n",
    "# for label, encoded_value in zip(label_encoder.classes_, label_encoder.transform(label_encoder.classes_)):\n",
    "#     print(f\"{label}: {encoded_value}\")\n",
    "\n",
    "# print(\"\\nEncoded DataFrame:\")\n",
    "# print(dataframe_copy[['Mibig_Product', 'Mibig_Product_Encoded']])\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Going Modular"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Overwriting python_scripts/json_csv.py\n"
     ]
    }
   ],
   "source": [
    "%%writefile python_scripts/json_csv.py \n",
    "import json\n",
    "import csv\n",
    "import os\n",
    "import argparse\n",
    "\n",
    "# Reads all of the json files in the dir\n",
    "def list_json_files(directory):\n",
    "    json_files = []\n",
    "    for root, dirs, files in os.walk(directory):\n",
    "        for file in files:\n",
    "            if file.endswith('.json'):\n",
    "                json_files.append(os.path.join(root, file))\n",
    "    return json_files\n",
    "\n",
    "def above_two(data_records):\n",
    "    index=[]\n",
    "    for i in range(len(data_records)):\n",
    "        modules = data_records[i][\"modules\"].keys()\n",
    "        if len(modules) > 2:\n",
    "            index.append(i)\n",
    "        else:\n",
    "            continue\n",
    "    return index\n",
    "\n",
    "def process_files(file_paths, csv_filename):\n",
    "    \n",
    "    # Check if the output file already exists\n",
    "    file_exists = os.path.isfile(csv_filename)\n",
    "    \n",
    "    with open(csv_filename, 'a', newline='') as csvfile:\n",
    "        writer = csv.writer(csvfile)\n",
    "        \n",
    "        if not file_exists:\n",
    "            writer.writerow([\"Bin_ID\", \"Contig_ID\", \"Gene_Coordinates\", \"Ctg_Number\", \"Ctg_Tag_Id\", \"Mibig_biosynthetic_gene_cluster\", \"Mibig_Product\", \"Blast_name\", \"Blast_genecluster\",\"Blast_annotation\",\"Blast_perc_coverage\", \"Blast_perc_ident\", \"Blast_blastscore\", \"Blast_evalue\", \"Blast_locus_tag\" ])\n",
    "        \n",
    "        for file_path in file_paths:\n",
    "            with open(file_path, 'r') as files:\n",
    "                data = json.load(files)\n",
    "                index_ = above_two(data[\"records\"])\n",
    "\n",
    "                for i in index_:\n",
    "                    bin_id = data[\"input_file\"]\n",
    "                    contig_id = data[\"records\"][i][\"modules\"][\"antismash.modules.clusterblast\"][\"record_id\"]\n",
    "                    blast_info = data[\"records\"][i][\"modules\"][\"antismash.modules.clusterblast\"][\"general\"][\"results\"][0][\"ranking\"][1][1][\"pairings\"]\n",
    "                    mibig_entries = data[\"records\"][i][\"modules\"][\"antismash.modules.clusterblast\"][\"knowncluster\"][\"mibig_entries\"][\"1\"]\n",
    "\n",
    "                    # Dict for each ctg_tagid\n",
    "                    blast_ = {}\n",
    "                    for blast in blast_info:\n",
    "                        ctg_name = blast[0].split(\"|\")[4]\n",
    "                        blast_[\"ctg_cordinates\"] = blast[0].split(\"|\")[2]\n",
    "                        blast_name, blast_genecluster, blast_start, blast_end, blast_strand, blast_annotation, blast_perc_ident, blast_blastscore, blast_perc_coverage, blast_evalue, blast_locus_tag  = blast[2].values()\n",
    "                        blast_[ctg_name] = {\"blast_name\" : blast_name, \n",
    "                                            \"blast_genecluster\": blast_genecluster,\n",
    "                                            \"blast_annotation\": blast_annotation,\n",
    "                                            \"blast_perc_coverage\":blast_perc_coverage,\n",
    "                                            \"blast_perc_ident\": blast_perc_ident,\n",
    "                                            \"blast_blastscore\": blast_blastscore,\n",
    "                                            \"blast_evalue\": blast_evalue,\n",
    "                                            \"blast_locus_tag\": blast_locus_tag,\n",
    "                                            \"cordinates\": blast[0].split(\"|\")[2] }\n",
    "\n",
    "                    for ctg_tagid, vl in mibig_entries.items():\n",
    "                        ctg_id = ctg_tagid.split(\"_\")[0]\n",
    "                        mibig_entry = vl[0]\n",
    "                        biosynthetic_gene_cluster_id = mibig_entry[2]\n",
    "                        mibig_entries_product = mibig_entry[4]\n",
    "                        blast_info_ = blast_.get(ctg_tagid)\n",
    "\n",
    "                        if blast_info_:\n",
    "                            ctg_cordinates = blast_info_[\"cordinates\"]\n",
    "                            blast_name = blast_info_[\"blast_name\"]\n",
    "                            blast_genecluster= blast_info_[\"blast_genecluster\"]\n",
    "                            blast_annotation = blast_info_[\"blast_annotation\"]\n",
    "                            blast_perc_coverage = blast_info_[\"blast_perc_coverage\"]\n",
    "                            blast_perc_ident = blast_info_[\"blast_perc_ident\"]\n",
    "                            blast_blastscore = blast_info_[\"blast_blastscore\"]\n",
    "                            blast_evalue = blast_info_[\"blast_evalue\"]\n",
    "                            blast_locus_tag = blast_info_[\"blast_locus_tag\"]\n",
    "\n",
    "                        else:\n",
    "                            ctg_cordinates = None\n",
    "                            blast_name = None\n",
    "                            blast_genecluster= None\n",
    "                            blast_annotation = None\n",
    "                            blast_perc_coverage = None\n",
    "                            blast_perc_ident = None\n",
    "                            blast_blastscore = None\n",
    "                            blast_evalue = None\n",
    "                            blast_locus_tag = None\n",
    "                            \n",
    "                        writer.writerow([\n",
    "                            bin_id, \n",
    "                            contig_id,\n",
    "                            ctg_cordinates, \n",
    "                            ctg_id, \n",
    "                            ctg_tagid, \n",
    "                            biosynthetic_gene_cluster_id, \n",
    "                            mibig_entries_product, \n",
    "                            blast_name,\n",
    "                            blast_genecluster,\n",
    "                            blast_annotation,\n",
    "                            blast_perc_coverage,\n",
    "                            blast_perc_ident,\n",
    "                            blast_blastscore,\n",
    "                            blast_evalue,\n",
    "                            blast_locus_tag\n",
    "                        ])\n",
    "            \n",
    "def main():\n",
    "    parser = argparse.ArgumentParser(description=\"Process JSON files and output combined results to a CSV file. Please note dont run with the same output filename it will add the same results in the file.\")\n",
    "    parser.add_argument('--result_dir',\"-r\", help='JSON files to process')\n",
    "    parser.add_argument('-o', '--output', default='combined_output.csv', help='Output CSV file name (default: combined_output.csv)')\n",
    "    \n",
    "    args = parser.parse_args()\n",
    "    \n",
    "    list_jsons = list_json_files(args.result_dir)\n",
    "\n",
    "    process_files(list_jsons, args.output)\n",
    "    print(f\"Output written to: {args.output}\")\n",
    "\n",
    "if __name__ == \"__main__\":\n",
    "    main()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Output written to: test_going_modular.csv\n"
     ]
    }
   ],
   "source": [
    "!python3 python_scripts/json_csv.py -r \"./results\" -o \"test_going_modular.csv\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "usage: json_csv.py [-h] [--result_dir RESULT_DIR] [-o OUTPUT]\n",
      "\n",
      "Process JSON files and output combined results to a CSV file. Please note dont\n",
      "run with the same output name file it will add the same results in the file.\n",
      "\n",
      "options:\n",
      "  -h, --help            show this help message and exit\n",
      "  --result_dir RESULT_DIR, -r RESULT_DIR\n",
      "                        JSON files to process\n",
      "  -o OUTPUT, --output OUTPUT\n",
      "                        Output CSV file name (default: combined_output.csv)\n"
     ]
    }
   ],
   "source": [
    "!python3 python_scripts/json_csv.py --help"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Bin_ID</th>\n",
       "      <th>Contig_ID</th>\n",
       "      <th>Gene_Coordinates</th>\n",
       "      <th>Ctg_Number</th>\n",
       "      <th>Ctg_Tag_Id</th>\n",
       "      <th>Mibig_biosynthetic_gene_cluster</th>\n",
       "      <th>Mibig_Product</th>\n",
       "      <th>Blast_name</th>\n",
       "      <th>Blast_genecluster</th>\n",
       "      <th>Blast_annotation</th>\n",
       "      <th>Blast_perc_coverage</th>\n",
       "      <th>Blast_perc_ident</th>\n",
       "      <th>Blast_blastscore</th>\n",
       "      <th>Blast_evalue</th>\n",
       "      <th>Blast_locus_tag</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_27</td>\n",
       "      <td>BGC0000588</td>\n",
       "      <td>RiPP:Microcin</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>28598-29438</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_28</td>\n",
       "      <td>BGC0000607</td>\n",
       "      <td>RiPP:Thiopeptide</td>\n",
       "      <td>CG392_RS04930</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>AAA_family_ATPase</td>\n",
       "      <td>99.641577</td>\n",
       "      <td>100.0</td>\n",
       "      <td>536.0</td>\n",
       "      <td>1.450000e-193</td>\n",
       "      <td>WP_004121099.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>43941-44757</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_38</td>\n",
       "      <td>BGC0002284</td>\n",
       "      <td>Other</td>\n",
       "      <td>CG392_RS04985</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>(p)ppGpp_synthetase</td>\n",
       "      <td>100.000000</td>\n",
       "      <td>95.0</td>\n",
       "      <td>493.0</td>\n",
       "      <td>8.280000e-177</td>\n",
       "      <td>WP_116438271.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>47230-48667</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_42</td>\n",
       "      <td>BGC0002561</td>\n",
       "      <td>Alkaloid</td>\n",
       "      <td>CG392_RS05005</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>hydroxymethylglutaryl-CoA_synthase</td>\n",
       "      <td>101.673640</td>\n",
       "      <td>84.0</td>\n",
       "      <td>765.0</td>\n",
       "      <td>4.340000e-278</td>\n",
       "      <td>WP_116438274.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>50447-51650</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_44</td>\n",
       "      <td>BGC0000867</td>\n",
       "      <td>Other</td>\n",
       "      <td>CG392_RS05015</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>acetyl-CoA_C-acyltransferase</td>\n",
       "      <td>106.500000</td>\n",
       "      <td>91.0</td>\n",
       "      <td>727.0</td>\n",
       "      <td>9.470000e-265</td>\n",
       "      <td>WP_116438276.1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "              Bin_ID                           Contig_ID Gene_Coordinates  \\\n",
       "0  MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542              NaN   \n",
       "1  MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      28598-29438   \n",
       "2  MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      43941-44757   \n",
       "3  MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      47230-48667   \n",
       "4  MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      50447-51650   \n",
       "\n",
       "  Ctg_Number Ctg_Tag_Id Mibig_biosynthetic_gene_cluster     Mibig_Product  \\\n",
       "0       ctg3    ctg3_27                      BGC0000588     RiPP:Microcin   \n",
       "1       ctg3    ctg3_28                      BGC0000607  RiPP:Thiopeptide   \n",
       "2       ctg3    ctg3_38                      BGC0002284             Other   \n",
       "3       ctg3    ctg3_42                      BGC0002561          Alkaloid   \n",
       "4       ctg3    ctg3_44                      BGC0000867             Other   \n",
       "\n",
       "      Blast_name              Blast_genecluster  \\\n",
       "0            NaN                            NaN   \n",
       "1  CG392_RS04930  NZ_NNRS01000004_c78906-120244   \n",
       "2  CG392_RS04985  NZ_NNRS01000004_c78906-120244   \n",
       "3  CG392_RS05005  NZ_NNRS01000004_c78906-120244   \n",
       "4  CG392_RS05015  NZ_NNRS01000004_c78906-120244   \n",
       "\n",
       "                     Blast_annotation  Blast_perc_coverage  Blast_perc_ident  \\\n",
       "0                                 NaN                  NaN               NaN   \n",
       "1                   AAA_family_ATPase            99.641577             100.0   \n",
       "2                 (p)ppGpp_synthetase           100.000000              95.0   \n",
       "3  hydroxymethylglutaryl-CoA_synthase           101.673640              84.0   \n",
       "4        acetyl-CoA_C-acyltransferase           106.500000              91.0   \n",
       "\n",
       "   Blast_blastscore   Blast_evalue Blast_locus_tag  \n",
       "0               NaN            NaN             NaN  \n",
       "1             536.0  1.450000e-193  WP_004121099.1  \n",
       "2             493.0  8.280000e-177  WP_116438271.1  \n",
       "3             765.0  4.340000e-278  WP_116438274.1  \n",
       "4             727.0  9.470000e-265  WP_116438276.1  "
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "results = pd.read_csv(\"./test_going_modular.csv\")\n",
    "\n",
    "results.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['MG_323829.bin2.fa', 'MG_323757.bin3.fa', 'MG_323829.bin1.fa',\n",
       "       'MG_323757.bin1.fa'], dtype=object)"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "results[\"Bin_ID\"].unique()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Going Modular Include Blast"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Overwriting python_scripts/json_csv_with_blast.py\n"
     ]
    }
   ],
   "source": [
    "%%writefile python_scripts/json_csv_with_blast.py \n",
    "import json\n",
    "import csv\n",
    "import os\n",
    "import argparse\n",
    "\n",
    "def above_two(data_records):\n",
    "    index=[]\n",
    "    for i in range(len(data_records)):\n",
    "        modules = data_records[i][\"modules\"].keys()\n",
    "        if len(modules) > 2:\n",
    "            # print(i, len(modules))\n",
    "            index.append(i)\n",
    "        else:\n",
    "            continue\n",
    "    return index\n",
    "\n",
    "# Reads all of the json files in the dir\n",
    "def list_json_files(directory):\n",
    "    json_files = []\n",
    "    for root, dirs, files in os.walk(directory):\n",
    "        for file in files:\n",
    "            if file.endswith('.json'):\n",
    "                json_files.append(os.path.join(root, file))\n",
    "    return json_files\n",
    "\n",
    "def process_files(file_paths, csv_filename):\n",
    "    \n",
    "    # Check if the output file already exists\n",
    "    file_exists = os.path.isfile(csv_filename)\n",
    "    \n",
    "    with open(csv_filename, 'a', newline='') as csvfile:\n",
    "        writer = csv.writer(csvfile)\n",
    "        \n",
    "        if not file_exists:\n",
    "            writer.writerow([\"Bin_ID\", \"Contig_ID\", \"Gene_Coordinates\", \"Ctg_Number\", \"Ctg_Tag_Id\", \"Mibig_biosynthetic_gene_cluster\", \"Mibig_Product\", \"Blast_name\", \"Blast_genecluster\",\"Blast_annotation\",\"Blast_perc_coverage\", \"Blast_perc_ident\", \"Blast_blastscore\", \"Blast_evalue\", \"Blast_locus_tag\" ])\n",
    "        \n",
    "        for file_path in file_paths:\n",
    "\n",
    "            with open(file_path, 'r') as files:\n",
    "                data = json.load(files)\n",
    "                index_ = above_two(data[\"records\"])\n",
    "                \n",
    "                for i in index_:\n",
    "                    bin_id = data[\"input_file\"]\n",
    "                    contig_id = data[\"records\"][i][\"modules\"][\"antismash.modules.clusterblast\"][\"record_id\"]\n",
    "                    blast_info = data[\"records\"][i][\"modules\"][\"antismash.modules.clusterblast\"][\"general\"][\"results\"][0][\"ranking\"][1][1][\"pairings\"]\n",
    "                    mibig_entries = data[\"records\"][i][\"modules\"][\"antismash.modules.clusterblast\"][\"knowncluster\"][\"mibig_entries\"][\"1\"]\n",
    "\n",
    "                    # Dict for each ctg_tagid\n",
    "                    blast_ = {}\n",
    "                    for blast in blast_info:\n",
    "                        ctg_name = blast[0].split(\"|\")[4]\n",
    "                        blast_[\"ctg_cordinates\"] = blast[0].split(\"|\")[2]\n",
    "                        blast_name, blast_genecluster, blast_start, blast_end, blast_strand, blast_annotation, blast_perc_ident, blast_blastscore, blast_perc_coverage, blast_evalue, blast_locus_tag  = blast[2].values()\n",
    "                        blast_[ctg_name] = {\"blast_name\" : blast_name, \n",
    "                                            \"blast_genecluster\": blast_genecluster,\n",
    "                                            \"blast_annotation\": blast_annotation,\n",
    "                                            \"blast_perc_coverage\":blast_perc_coverage,\n",
    "                                            \"blast_perc_ident\": blast_perc_ident,\n",
    "                                            \"blast_blastscore\": blast_blastscore,\n",
    "                                            \"blast_evalue\": blast_evalue,\n",
    "                                            \"blast_locus_tag\": blast_locus_tag,\n",
    "                                            \"cordinates\": blast[0].split(\"|\")[2] }\n",
    "\n",
    "                    blast_keys = list(blast_.keys())[1:]\n",
    "                    mibig_entries_keys = [ ctg_tagid for ctg_tagid in mibig_entries.keys()]\n",
    "        \n",
    "\n",
    "                    for ix, vl in  enumerate(blast_keys):\n",
    "                        ctg_id = vl.split(\"_\")[0]\n",
    "                        ctg_tagid = vl\n",
    "                        ctg_cordinates = blast_[vl][\"cordinates\"]\n",
    "                        blast_name = blast_[vl][\"blast_name\"]\n",
    "                        blast_genecluster= blast_[vl][\"blast_genecluster\"]\n",
    "                        blast_annotation = blast_[vl][\"blast_annotation\"]\n",
    "                        blast_perc_coverage = blast_[vl][\"blast_perc_coverage\"]\n",
    "                        blast_perc_ident = blast_[vl][\"blast_perc_ident\"]\n",
    "                        blast_blastscore = blast_[vl][\"blast_blastscore\"]\n",
    "                        blast_evalue = blast_[vl][\"blast_evalue\"]\n",
    "                        blast_locus_tag = blast_[vl][\"blast_locus_tag\"]\n",
    "                        biosynthetic_gene_cluster_id=None \n",
    "                        mibig_entries_product=None\n",
    "\n",
    "                        if vl not in mibig_entries_keys:\n",
    "                            writer.writerow([\n",
    "                                    bin_id, \n",
    "                                    contig_id,\n",
    "                                    ctg_cordinates, \n",
    "                                    ctg_id, \n",
    "                                    ctg_tagid, \n",
    "                                    biosynthetic_gene_cluster_id, \n",
    "                                    mibig_entries_product, \n",
    "                                    blast_name,\n",
    "                                    blast_genecluster,\n",
    "                                    blast_annotation,\n",
    "                                    blast_perc_coverage,\n",
    "                                    blast_perc_ident,\n",
    "                                    blast_blastscore,\n",
    "                                    blast_evalue,\n",
    "                                    blast_locus_tag\n",
    "                                ])\n",
    "\n",
    "\n",
    "\n",
    "                    for ctg_tagid, vl in mibig_entries.items():\n",
    "                        ctg_id = ctg_tagid.split(\"_\")[0]\n",
    "                        mibig_entry = vl[0]\n",
    "                        biosynthetic_gene_cluster_id = mibig_entry[2]\n",
    "                        mibig_entries_product = mibig_entry[4]\n",
    "                        blast_info_ = blast_.get(ctg_tagid)\n",
    "\n",
    "                        if blast_info_:\n",
    "                            ctg_cordinates = blast_info_[\"cordinates\"]\n",
    "                            blast_name = blast_info_[\"blast_name\"]\n",
    "                            blast_genecluster= blast_info_[\"blast_genecluster\"]\n",
    "                            blast_annotation = blast_info_[\"blast_annotation\"]\n",
    "                            blast_perc_coverage = blast_info_[\"blast_perc_coverage\"]\n",
    "                            blast_perc_ident = blast_info_[\"blast_perc_ident\"]\n",
    "                            blast_blastscore = blast_info_[\"blast_blastscore\"]\n",
    "                            blast_evalue = blast_info_[\"blast_evalue\"]\n",
    "                            blast_locus_tag = blast_info_[\"blast_locus_tag\"]\n",
    "\n",
    "                        else:\n",
    "                            ctg_cordinates = None\n",
    "                            blast_name = None\n",
    "                            blast_genecluster= None\n",
    "                            blast_annotation = None\n",
    "                            blast_perc_coverage = None\n",
    "                            blast_perc_ident = None\n",
    "                            blast_blastscore = None\n",
    "                            blast_evalue = None\n",
    "                            blast_locus_tag = None\n",
    "                            \n",
    "                        writer.writerow([\n",
    "                            bin_id, \n",
    "                            contig_id,\n",
    "                            ctg_cordinates, \n",
    "                            ctg_id, \n",
    "                            ctg_tagid, \n",
    "                            biosynthetic_gene_cluster_id, \n",
    "                            mibig_entries_product, \n",
    "                            blast_name,\n",
    "                            blast_genecluster,\n",
    "                            blast_annotation,\n",
    "                            blast_perc_coverage,\n",
    "                            blast_perc_ident,\n",
    "                            blast_blastscore,\n",
    "                            blast_evalue,\n",
    "                            blast_locus_tag\n",
    "                        ])\n",
    "\n",
    "def main():\n",
    "    parser = argparse.ArgumentParser(description=\"Process JSON files and output combined results to a CSV file. Please note dont run with the same output filename it will add the same results in the file.\")\n",
    "    parser.add_argument('--result_dir',\"-r\", help='JSON files to process')\n",
    "    parser.add_argument('-o', '--output', default='combined_output.csv', help='Output CSV file name (default: combined_output.csv)')\n",
    "    \n",
    "    args = parser.parse_args()\n",
    "    \n",
    "    list_jsons = list_json_files(args.result_dir)\n",
    "\n",
    "    process_files(list_jsons, args.output)\n",
    "    print(f\"Output written to: {args.output}\")\n",
    "\n",
    "if __name__ == \"__main__\":\n",
    "    main()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Output written to: output_with_blast.csv\n"
     ]
    }
   ],
   "source": [
    "!python3 python_scripts/json_csv_with_blast.py -r \"./results\" -o \"output_with_blast.csv\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Bin_ID</th>\n",
       "      <th>Contig_ID</th>\n",
       "      <th>Gene_Coordinates</th>\n",
       "      <th>Ctg_Number</th>\n",
       "      <th>Ctg_Tag_Id</th>\n",
       "      <th>Mibig_biosynthetic_gene_cluster</th>\n",
       "      <th>Mibig_Product</th>\n",
       "      <th>Blast_name</th>\n",
       "      <th>Blast_genecluster</th>\n",
       "      <th>Blast_annotation</th>\n",
       "      <th>Blast_perc_coverage</th>\n",
       "      <th>Blast_perc_ident</th>\n",
       "      <th>Blast_blastscore</th>\n",
       "      <th>Blast_evalue</th>\n",
       "      <th>Blast_locus_tag</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>29585-29864</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_29</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CG392_RS04935</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>hypothetical_protein</td>\n",
       "      <td>98.913043</td>\n",
       "      <td>100.0</td>\n",
       "      <td>186.0</td>\n",
       "      <td>2.970000e-61</td>\n",
       "      <td>WP_116438264.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>29958-30246</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_30</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CG392_RS04940</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>type_II_toxin-antitoxin_system_RelB/DinJ_famil...</td>\n",
       "      <td>100.000000</td>\n",
       "      <td>100.0</td>\n",
       "      <td>192.0</td>\n",
       "      <td>1.900000e-63</td>\n",
       "      <td>WP_116438265.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>30616-31054</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_31</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CG392_RS04945</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>ORF6N_domain-containing_protein</td>\n",
       "      <td>79.310345</td>\n",
       "      <td>74.0</td>\n",
       "      <td>160.0</td>\n",
       "      <td>5.670000e-50</td>\n",
       "      <td>CG392_RS04945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>31058-32072</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_32</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CG392_RS04950</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>Abi_family_protein</td>\n",
       "      <td>100.000000</td>\n",
       "      <td>100.0</td>\n",
       "      <td>679.0</td>\n",
       "      <td>2.570000e-248</td>\n",
       "      <td>WP_116438267.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>MG_323829.bin2.fa</td>\n",
       "      <td>NODE_3_length_155941_cov_99.724542</td>\n",
       "      <td>33418-38701</td>\n",
       "      <td>ctg3</td>\n",
       "      <td>ctg3_34</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CG392_RS07955</td>\n",
       "      <td>NZ_NNRS01000004_c78906-120244</td>\n",
       "      <td>hypothetical_protein</td>\n",
       "      <td>45.397727</td>\n",
       "      <td>45.0</td>\n",
       "      <td>571.0</td>\n",
       "      <td>6.240000e-181</td>\n",
       "      <td>CG392_RS07955</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>MG_323757.bin1.fa</td>\n",
       "      <td>NODE_83_length_8923_cov_7.096640</td>\n",
       "      <td>813-996</td>\n",
       "      <td>ctg59</td>\n",
       "      <td>ctg59_2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>KW861_RS14645</td>\n",
       "      <td>NZ_CP078505_c3139533-3154876</td>\n",
       "      <td>Blp_family_class_II_bacteriocin</td>\n",
       "      <td>100.000000</td>\n",
       "      <td>83.0</td>\n",
       "      <td>80.0</td>\n",
       "      <td>5.870000e-20</td>\n",
       "      <td>WP_103301327.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>MG_323757.bin1.fa</td>\n",
       "      <td>NODE_83_length_8923_cov_7.096640</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ctg59</td>\n",
       "      <td>ctg59_6</td>\n",
       "      <td>BGC0000100</td>\n",
       "      <td>Polyketide</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>MG_323757.bin1.fa</td>\n",
       "      <td>NODE_110_length_7016_cov_6.405832</td>\n",
       "      <td>520-1279</td>\n",
       "      <td>ctg82</td>\n",
       "      <td>ctg82_1</td>\n",
       "      <td>BGC0002667</td>\n",
       "      <td>RiPP</td>\n",
       "      <td>A5816_RS11485</td>\n",
       "      <td>NZ_NGLI01000002_c555810-572363</td>\n",
       "      <td>LytTR_family_DNA-binding_domain-containing_pro...</td>\n",
       "      <td>98.809524</td>\n",
       "      <td>93.0</td>\n",
       "      <td>460.0</td>\n",
       "      <td>1.940000e-164</td>\n",
       "      <td>WP_010733065.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>MG_323757.bin1.fa</td>\n",
       "      <td>NODE_110_length_7016_cov_6.405832</td>\n",
       "      <td>1280-2594</td>\n",
       "      <td>ctg82</td>\n",
       "      <td>ctg82_2</td>\n",
       "      <td>BGC0000624</td>\n",
       "      <td>RiPP</td>\n",
       "      <td>A5816_RS11480</td>\n",
       "      <td>NZ_NGLI01000002_c555810-572363</td>\n",
       "      <td>GHKL_domain-containing_protein</td>\n",
       "      <td>72.311213</td>\n",
       "      <td>88.0</td>\n",
       "      <td>527.0</td>\n",
       "      <td>6.040000e-187</td>\n",
       "      <td>WP_080114483.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>MG_323757.bin1.fa</td>\n",
       "      <td>NODE_110_length_7016_cov_6.405832</td>\n",
       "      <td>2682-4854</td>\n",
       "      <td>ctg82</td>\n",
       "      <td>ctg82_3</td>\n",
       "      <td>BGC0002579</td>\n",
       "      <td>RiPP</td>\n",
       "      <td>A5816_RS11475</td>\n",
       "      <td>NZ_NGLI01000002_c555810-572363</td>\n",
       "      <td>peptide_cleavage/export_ABC_transporter</td>\n",
       "      <td>100.000000</td>\n",
       "      <td>91.0</td>\n",
       "      <td>1263.0</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>WP_010733063.1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>94 rows Ã— 15 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "               Bin_ID                           Contig_ID Gene_Coordinates  \\\n",
       "0   MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      29585-29864   \n",
       "1   MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      29958-30246   \n",
       "2   MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      30616-31054   \n",
       "3   MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      31058-32072   \n",
       "4   MG_323829.bin2.fa  NODE_3_length_155941_cov_99.724542      33418-38701   \n",
       "..                ...                                 ...              ...   \n",
       "89  MG_323757.bin1.fa    NODE_83_length_8923_cov_7.096640          813-996   \n",
       "90  MG_323757.bin1.fa    NODE_83_length_8923_cov_7.096640              NaN   \n",
       "91  MG_323757.bin1.fa   NODE_110_length_7016_cov_6.405832         520-1279   \n",
       "92  MG_323757.bin1.fa   NODE_110_length_7016_cov_6.405832        1280-2594   \n",
       "93  MG_323757.bin1.fa   NODE_110_length_7016_cov_6.405832        2682-4854   \n",
       "\n",
       "   Ctg_Number Ctg_Tag_Id Mibig_biosynthetic_gene_cluster Mibig_Product  \\\n",
       "0        ctg3    ctg3_29                             NaN           NaN   \n",
       "1        ctg3    ctg3_30                             NaN           NaN   \n",
       "2        ctg3    ctg3_31                             NaN           NaN   \n",
       "3        ctg3    ctg3_32                             NaN           NaN   \n",
       "4        ctg3    ctg3_34                             NaN           NaN   \n",
       "..        ...        ...                             ...           ...   \n",
       "89      ctg59    ctg59_2                             NaN           NaN   \n",
       "90      ctg59    ctg59_6                      BGC0000100    Polyketide   \n",
       "91      ctg82    ctg82_1                      BGC0002667          RiPP   \n",
       "92      ctg82    ctg82_2                      BGC0000624          RiPP   \n",
       "93      ctg82    ctg82_3                      BGC0002579          RiPP   \n",
       "\n",
       "       Blast_name               Blast_genecluster  \\\n",
       "0   CG392_RS04935   NZ_NNRS01000004_c78906-120244   \n",
       "1   CG392_RS04940   NZ_NNRS01000004_c78906-120244   \n",
       "2   CG392_RS04945   NZ_NNRS01000004_c78906-120244   \n",
       "3   CG392_RS04950   NZ_NNRS01000004_c78906-120244   \n",
       "4   CG392_RS07955   NZ_NNRS01000004_c78906-120244   \n",
       "..            ...                             ...   \n",
       "89  KW861_RS14645    NZ_CP078505_c3139533-3154876   \n",
       "90            NaN                             NaN   \n",
       "91  A5816_RS11485  NZ_NGLI01000002_c555810-572363   \n",
       "92  A5816_RS11480  NZ_NGLI01000002_c555810-572363   \n",
       "93  A5816_RS11475  NZ_NGLI01000002_c555810-572363   \n",
       "\n",
       "                                     Blast_annotation  Blast_perc_coverage  \\\n",
       "0                                hypothetical_protein            98.913043   \n",
       "1   type_II_toxin-antitoxin_system_RelB/DinJ_famil...           100.000000   \n",
       "2                     ORF6N_domain-containing_protein            79.310345   \n",
       "3                                  Abi_family_protein           100.000000   \n",
       "4                                hypothetical_protein            45.397727   \n",
       "..                                                ...                  ...   \n",
       "89                    Blp_family_class_II_bacteriocin           100.000000   \n",
       "90                                                NaN                  NaN   \n",
       "91  LytTR_family_DNA-binding_domain-containing_pro...            98.809524   \n",
       "92                     GHKL_domain-containing_protein            72.311213   \n",
       "93            peptide_cleavage/export_ABC_transporter           100.000000   \n",
       "\n",
       "    Blast_perc_ident  Blast_blastscore   Blast_evalue Blast_locus_tag  \n",
       "0              100.0             186.0   2.970000e-61  WP_116438264.1  \n",
       "1              100.0             192.0   1.900000e-63  WP_116438265.1  \n",
       "2               74.0             160.0   5.670000e-50   CG392_RS04945  \n",
       "3              100.0             679.0  2.570000e-248  WP_116438267.1  \n",
       "4               45.0             571.0  6.240000e-181   CG392_RS07955  \n",
       "..               ...               ...            ...             ...  \n",
       "89              83.0              80.0   5.870000e-20  WP_103301327.1  \n",
       "90               NaN               NaN            NaN             NaN  \n",
       "91              93.0             460.0  1.940000e-164  WP_010733065.1  \n",
       "92              88.0             527.0  6.040000e-187  WP_080114483.1  \n",
       "93              91.0            1263.0   0.000000e+00  WP_010733063.1  \n",
       "\n",
       "[94 rows x 15 columns]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "df= pd.read_csv(\"./output_with_blast.csv\")\n",
    "\n",
    "df.head(1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(94, 15)"
      ]
     },
     "execution_count": 44,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Add Taxanom"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Successfully converted MAG_inventory_VIRGO2.txt to MAG_inventory_VIRGO2.csv\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import csv\n",
    "import sys\n",
    "\n",
    "def convert_txt_to_csv(input_file, output_file):\n",
    "    try:\n",
    "        with open(input_file, 'r', newline='') as infile, open(output_file, 'w', newline='') as outfile:\n",
    "            reader = csv.reader(infile, delimiter='\\t')\n",
    "            writer = csv.writer(outfile)\n",
    "\n",
    "            # Write all rows to the CSV file\n",
    "            for row in reader:\n",
    "                writer.writerow(row)\n",
    "\n",
    "        print(f\"Successfully converted {input_file} to {output_file}\")\n",
    "    except Exception as e:\n",
    "        print(f\"An error occurred: {e}\")\n",
    "        sys.exit(1)\n",
    "\n",
    "if __name__ == \"__main__\":\n",
    "    input_file = \"MAG_inventory_VIRGO2.txt\" \n",
    "    output_file = \"MAG_inventory_VIRGO2.csv\"  \n",
    "\n",
    "    convert_txt_to_csv(input_file, output_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>MAG</th>\n",
       "      <th>MAG_originalName</th>\n",
       "      <th>Metagenome</th>\n",
       "      <th>Project</th>\n",
       "      <th>Taxonomy</th>\n",
       "      <th>FilterContam</th>\n",
       "      <th>KeepCleaned</th>\n",
       "      <th>R1Complete</th>\n",
       "      <th>R2Complete</th>\n",
       "      <th>ComplDif</th>\n",
       "      <th>...</th>\n",
       "      <th>closest_placement_ani</th>\n",
       "      <th>closest_placement_af</th>\n",
       "      <th>pplacer_taxonomy</th>\n",
       "      <th>classification_method</th>\n",
       "      <th>note</th>\n",
       "      <th>other_related_references(genome_id,species_name,radius,ANI,AF)</th>\n",
       "      <th>msa_percent</th>\n",
       "      <th>translation_table</th>\n",
       "      <th>red_value</th>\n",
       "      <th>warnings</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MAG00001</td>\n",
       "      <td>MG_0052RFLQV1.bin1</td>\n",
       "      <td>MG_0052RFLQV1</td>\n",
       "      <td>XPENN</td>\n",
       "      <td>Lactobacillus_crispatus</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>98.86</td>\n",
       "      <td>98.86</td>\n",
       "      <td>0.00</td>\n",
       "      <td>...</td>\n",
       "      <td>97.56</td>\n",
       "      <td>0.70</td>\n",
       "      <td>d__Bacteria;p__Firmicutes;c__Bacilli;o__Lactob...</td>\n",
       "      <td>taxonomic classification defined by topology a...</td>\n",
       "      <td>topological placement and ANI have congruent s...</td>\n",
       "      <td>GCF_900103655.1, s__Lactobacillus kefiranofaci...</td>\n",
       "      <td>90.27</td>\n",
       "      <td>11</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>MAG00002</td>\n",
       "      <td>MG_0139RFLQV1.bin1</td>\n",
       "      <td>MG_0139RFLQV1</td>\n",
       "      <td>XPENN</td>\n",
       "      <td>Lactobacillus_mulieris</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>87.28</td>\n",
       "      <td>85.74</td>\n",
       "      <td>-1.54</td>\n",
       "      <td>...</td>\n",
       "      <td>96.99</td>\n",
       "      <td>0.91</td>\n",
       "      <td>d__Bacteria;p__Firmicutes;c__Bacilli;o__Lactob...</td>\n",
       "      <td>taxonomic classification defined by topology a...</td>\n",
       "      <td>topological placement and ANI have congruent s...</td>\n",
       "      <td>GCF_001436455.1, s__Lactobacillus jensenii, 95...</td>\n",
       "      <td>87.43</td>\n",
       "      <td>11</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>MAG00003</td>\n",
       "      <td>MG_0139RFLQV1.bin2</td>\n",
       "      <td>MG_0139RFLQV1</td>\n",
       "      <td>XPENN</td>\n",
       "      <td>Bradyrhizobium_sp003020075</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>80.75</td>\n",
       "      <td>79.86</td>\n",
       "      <td>-0.89</td>\n",
       "      <td>...</td>\n",
       "      <td>99.77</td>\n",
       "      <td>0.96</td>\n",
       "      <td>d__Bacteria;p__Proteobacteria;c__Alphaproteoba...</td>\n",
       "      <td>taxonomic classification defined by topology a...</td>\n",
       "      <td>topological placement and ANI have congruent s...</td>\n",
       "      <td>GCA_001238275.1, s__Bradyrhizobium viridifutur...</td>\n",
       "      <td>62.71</td>\n",
       "      <td>11</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>MAG00004</td>\n",
       "      <td>MG_0139RFLQV1.bin3</td>\n",
       "      <td>MG_0139RFLQV1</td>\n",
       "      <td>XPENN</td>\n",
       "      <td>Lactobacillus_iners</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>94.01</td>\n",
       "      <td>94.02</td>\n",
       "      <td>0.01</td>\n",
       "      <td>...</td>\n",
       "      <td>98.73</td>\n",
       "      <td>0.94</td>\n",
       "      <td>d__Bacteria;p__Firmicutes;c__Bacilli;o__Lactob...</td>\n",
       "      <td>taxonomic classification defined by topology a...</td>\n",
       "      <td>topological placement and ANI have congruent s...</td>\n",
       "      <td>GCF_003423665.1, s__Lactobacillus rodentium, 9...</td>\n",
       "      <td>91.32</td>\n",
       "      <td>11</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>MAG00005</td>\n",
       "      <td>MG_0226RFLQV1.bin1</td>\n",
       "      <td>MG_0226RFLQV1</td>\n",
       "      <td>XPENN</td>\n",
       "      <td>Cutibacterium_modestum</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>83.47</td>\n",
       "      <td>82.60</td>\n",
       "      <td>-0.87</td>\n",
       "      <td>...</td>\n",
       "      <td>99.91</td>\n",
       "      <td>1.00</td>\n",
       "      <td>d__Bacteria;p__Actinobacteriota;c__Actinomycet...</td>\n",
       "      <td>taxonomic classification defined by topology a...</td>\n",
       "      <td>topological placement and ANI have congruent s...</td>\n",
       "      <td>GCF_003030305.1, s__Cutibacterium acnes, 95.0,...</td>\n",
       "      <td>62.59</td>\n",
       "      <td>11</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 35 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        MAG    MAG_originalName     Metagenome Project  \\\n",
       "0  MAG00001  MG_0052RFLQV1.bin1  MG_0052RFLQV1   XPENN   \n",
       "1  MAG00002  MG_0139RFLQV1.bin1  MG_0139RFLQV1   XPENN   \n",
       "2  MAG00003  MG_0139RFLQV1.bin2  MG_0139RFLQV1   XPENN   \n",
       "3  MAG00004  MG_0139RFLQV1.bin3  MG_0139RFLQV1   XPENN   \n",
       "4  MAG00005  MG_0226RFLQV1.bin1  MG_0226RFLQV1   XPENN   \n",
       "\n",
       "                     Taxonomy  FilterContam  KeepCleaned  R1Complete  \\\n",
       "0     Lactobacillus_crispatus           NaN            0       98.86   \n",
       "1      Lactobacillus_mulieris           NaN            0       87.28   \n",
       "2  Bradyrhizobium_sp003020075           1.0            0       80.75   \n",
       "3         Lactobacillus_iners           NaN            0       94.01   \n",
       "4      Cutibacterium_modestum           NaN            1       83.47   \n",
       "\n",
       "   R2Complete  ComplDif  ...  closest_placement_ani  closest_placement_af  \\\n",
       "0       98.86      0.00  ...                  97.56                  0.70   \n",
       "1       85.74     -1.54  ...                  96.99                  0.91   \n",
       "2       79.86     -0.89  ...                  99.77                  0.96   \n",
       "3       94.02      0.01  ...                  98.73                  0.94   \n",
       "4       82.60     -0.87  ...                  99.91                  1.00   \n",
       "\n",
       "                                    pplacer_taxonomy  \\\n",
       "0  d__Bacteria;p__Firmicutes;c__Bacilli;o__Lactob...   \n",
       "1  d__Bacteria;p__Firmicutes;c__Bacilli;o__Lactob...   \n",
       "2  d__Bacteria;p__Proteobacteria;c__Alphaproteoba...   \n",
       "3  d__Bacteria;p__Firmicutes;c__Bacilli;o__Lactob...   \n",
       "4  d__Bacteria;p__Actinobacteriota;c__Actinomycet...   \n",
       "\n",
       "                               classification_method  \\\n",
       "0  taxonomic classification defined by topology a...   \n",
       "1  taxonomic classification defined by topology a...   \n",
       "2  taxonomic classification defined by topology a...   \n",
       "3  taxonomic classification defined by topology a...   \n",
       "4  taxonomic classification defined by topology a...   \n",
       "\n",
       "                                                note  \\\n",
       "0  topological placement and ANI have congruent s...   \n",
       "1  topological placement and ANI have congruent s...   \n",
       "2  topological placement and ANI have congruent s...   \n",
       "3  topological placement and ANI have congruent s...   \n",
       "4  topological placement and ANI have congruent s...   \n",
       "\n",
       "   other_related_references(genome_id,species_name,radius,ANI,AF) msa_percent  \\\n",
       "0  GCF_900103655.1, s__Lactobacillus kefiranofaci...                    90.27   \n",
       "1  GCF_001436455.1, s__Lactobacillus jensenii, 95...                    87.43   \n",
       "2  GCA_001238275.1, s__Bradyrhizobium viridifutur...                    62.71   \n",
       "3  GCF_003423665.1, s__Lactobacillus rodentium, 9...                    91.32   \n",
       "4  GCF_003030305.1, s__Cutibacterium acnes, 95.0,...                    62.59   \n",
       "\n",
       "  translation_table  red_value warnings  \n",
       "0                11        NaN      NaN  \n",
       "1                11        NaN      NaN  \n",
       "2                11        NaN      NaN  \n",
       "3                11        NaN      NaN  \n",
       "4                11        NaN      NaN  \n",
       "\n",
       "[5 rows x 35 columns]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mag_virgo = pd.read_csv(\"MAG_inventory_VIRGO2.csv\")\n",
    "mag_virgo.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Create new column MAG_originalName on the json results dataframe\n",
    "df[\"MAG_originalName\"] = df[\"Bin_ID\"].map(lambda x: x.split(\".\")[0]+ \".\" + x.split(\".\")[1])\n",
    "\n",
    "# Merge the MAG_inventory_VIRGO2 on MAG_originalName with the json results\n",
    "df_results_tax =  pd.merge(df, mag_virgo[['MAG_originalName', 'Taxonomy']], on='MAG_originalName', how='inner')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['Gardnerella_piotii', 'Lactobacillus_iners',\n",
       "       'Streptococcus_agalactiae', 'Lactobacillus_mulieris'], dtype=object)"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_results_tax.Taxonomy.unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Taxonomy</th>\n",
       "      <th>Mibig_Product</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Gardnerella_piotii</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Gardnerella_piotii</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Gardnerella_piotii</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Gardnerella_piotii</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Gardnerella_piotii</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>Lactobacillus_mulieris</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>Lactobacillus_mulieris</td>\n",
       "      <td>Polyketide</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>Lactobacillus_mulieris</td>\n",
       "      <td>RiPP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>Lactobacillus_mulieris</td>\n",
       "      <td>RiPP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>Lactobacillus_mulieris</td>\n",
       "      <td>RiPP</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>94 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  Taxonomy Mibig_Product\n",
       "0       Gardnerella_piotii           NaN\n",
       "1       Gardnerella_piotii           NaN\n",
       "2       Gardnerella_piotii           NaN\n",
       "3       Gardnerella_piotii           NaN\n",
       "4       Gardnerella_piotii           NaN\n",
       "..                     ...           ...\n",
       "89  Lactobacillus_mulieris           NaN\n",
       "90  Lactobacillus_mulieris    Polyketide\n",
       "91  Lactobacillus_mulieris          RiPP\n",
       "92  Lactobacillus_mulieris          RiPP\n",
       "93  Lactobacillus_mulieris          RiPP\n",
       "\n",
       "[94 rows x 2 columns]"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_results_tax[[\"Taxonomy\", \"Mibig_Product\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
