
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>MG_323829.bin1 - 2 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="MG_323829.bin1 - 2 record(s) - 2 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/MG_323829.bin1/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.1.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="MG_323829.bin1.zip">Download all results</a></li>

<li><a href="MG_323829.bin1.gbk">Download GenBank summary file</a></li>
<li><a href="MG_323829.bin1.json">Download JSON results file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton PKS arylpolyene r2c1">
<a href="#r2c1">2.1</a>
</div>



<div class="regbutton PKS T3PKS r47c1">
<a href="#r47c1">47.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NODE_7_length_101522_cov_12.966659</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="NODE_7_length_101522_cov_12.966659_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton PKS arylpolyene r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#arylpolyene" target="_blank">arylpolyene</a>
</td>
<td class="digits">5,460</td>
<td class="digits table-split-left">47,655</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_125_length_11688_cov_13.230809</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-47" data-filename="NODE_125_length_11688_cov_13.230809_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-47" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r47c1">
<td class="regbutton PKS T3PKS r47c1">
<a href="#r47c1">Region&nbsp47.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">11,688</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton PKS arylpolyene r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#arylpolyene" target="_blank">arylpolyene</a>
</td>
<td class="digits">5,460</td>
<td class="digits table-split-left">47,655</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r47c1">
<td class="regbutton PKS T3PKS r47c1">
<a href="#r47c1">Region&nbsp47.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">11,688</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NODE_4_length_121126_cov_14.544094</strong></dd>

<dd><strong>NODE_8_length_92599_cov_14.451148</strong></dd>

<dd><strong>NODE_9_length_88868_cov_13.609382</strong></dd>

<dd><strong>NODE_13_length_79491_cov_13.445226</strong></dd>

<dd><strong>NODE_16_length_73408_cov_12.954071</strong></dd>

<dd><strong>NODE_18_length_64318_cov_12.796819</strong></dd>

<dd><strong>NODE_19_length_63879_cov_14.886908</strong></dd>

<dd><strong>NODE_22_length_57729_cov_19.640670</strong></dd>

<dd><strong>NODE_24_length_55835_cov_12.261294</strong></dd>

<dd><strong>NODE_26_length_55371_cov_13.011931</strong></dd>

<dd><strong>NODE_27_length_52070_cov_12.906988</strong></dd>

<dd><strong>NODE_29_length_48356_cov_13.214302</strong></dd>

<dd><strong>NODE_32_length_44263_cov_15.330393</strong></dd>

<dd><strong>NODE_33_length_44202_cov_12.319976</strong></dd>

<dd><strong>NODE_34_length_42638_cov_15.452152</strong></dd>

<dd><strong>NODE_38_length_39157_cov_18.161961</strong></dd>

<dd><strong>NODE_39_length_38989_cov_16.415806</strong></dd>

<dd><strong>NODE_41_length_36406_cov_15.189128</strong></dd>

<dd><strong>NODE_43_length_35000_cov_15.608041</strong></dd>

<dd><strong>NODE_48_length_31460_cov_17.252699</strong></dd>

<dd><strong>NODE_51_length_30530_cov_13.619820</strong></dd>

<dd><strong>NODE_52_length_29879_cov_13.110481</strong></dd>

<dd><strong>NODE_56_length_29085_cov_12.755425</strong></dd>

<dd><strong>NODE_57_length_29066_cov_13.110303</strong></dd>

<dd><strong>NODE_58_length_28786_cov_14.821760</strong></dd>

<dd><strong>NODE_60_length_27661_cov_15.987974</strong></dd>

<dd><strong>NODE_63_length_27088_cov_15.422558</strong></dd>

<dd><strong>NODE_64_length_26947_cov_11.643797</strong></dd>

<dd><strong>NODE_74_length_22255_cov_13.871712</strong></dd>

<dd><strong>NODE_77_length_21521_cov_17.504379</strong></dd>

<dd><strong>NODE_79_length_21419_cov_16.295076</strong></dd>

<dd><strong>NODE_80_length_21337_cov_12.906635</strong></dd>

<dd><strong>NODE_82_length_19624_cov_13.374265</strong></dd>

<dd><strong>NODE_83_length_19399_cov_13.168114</strong></dd>

<dd><strong>NODE_85_length_19075_cov_15.844690</strong></dd>

<dd><strong>NODE_86_length_18992_cov_15.915562</strong></dd>

<dd><strong>NODE_88_length_18796_cov_15.853156</strong></dd>

<dd><strong>NODE_92_length_18032_cov_14.438004</strong></dd>

<dd><strong>NODE_98_length_15628_cov_12.161690</strong></dd>

<dd><strong>NODE_99_length_15610_cov_19.139184</strong></dd>

<dd><strong>NODE_106_length_15027_cov_10.314721</strong></dd>

<dd><strong>NODE_113_length_13503_cov_10.658760</strong></dd>

<dd><strong>NODE_114_length_13491_cov_9.900417</strong></dd>

<dd><strong>NODE_121_length_12251_cov_15.570351</strong></dd>

<dd><strong>NODE_122_length_11982_cov_13.431961</strong></dd>

<dd><strong>NODE_127_length_11610_cov_15.500736</strong></dd>

<dd><strong>NODE_132_length_11153_cov_24.335376</strong></dd>

<dd><strong>NODE_133_length_10750_cov_11.707246</strong></dd>

<dd><strong>NODE_143_length_10348_cov_17.964539</strong></dd>

<dd><strong>NODE_148_length_9820_cov_17.224066</strong></dd>

<dd><strong>NODE_162_length_8678_cov_19.704511</strong></dd>

<dd><strong>NODE_163_length_8646_cov_13.185892</strong></dd>

<dd><strong>NODE_166_length_8226_cov_11.434096</strong></dd>

<dd><strong>NODE_172_length_7778_cov_12.571669</strong></dd>

<dd><strong>NODE_174_length_7750_cov_14.579727</strong></dd>

<dd><strong>NODE_184_length_7282_cov_19.901619</strong></dd>

<dd><strong>NODE_187_length_7193_cov_13.939619</strong></dd>

<dd><strong>NODE_193_length_6799_cov_15.452254</strong></dd>

<dd><strong>NODE_197_length_6743_cov_8.537679</strong></dd>

<dd><strong>NODE_205_length_6361_cov_14.293371</strong></dd>

<dd><strong>NODE_213_length_5916_cov_9.500768</strong></dd>

<dd><strong>NODE_222_length_5647_cov_11.501788</strong></dd>

<dd><strong>NODE_238_length_5199_cov_12.326400</strong></dd>

<dd><strong>NODE_248_length_4828_cov_15.539912</strong></dd>

<dd><strong>NODE_250_length_4756_cov_9.525420</strong></dd>

<dd><strong>NODE_253_length_4679_cov_15.015355</strong></dd>

<dd><strong>NODE_256_length_4604_cov_14.835348</strong></dd>

<dd><strong>NODE_265_length_4406_cov_13.376695</strong></dd>

<dd><strong>NODE_269_length_4331_cov_13.307530</strong></dd>

<dd><strong>NODE_285_length_4079_cov_12.454771</strong></dd>

<dd><strong>NODE_289_length_3984_cov_12.463477</strong></dd>

<dd><strong>NODE_291_length_3964_cov_10.852648</strong></dd>

<dd><strong>NODE_293_length_3963_cov_9.461105</strong></dd>

<dd><strong>NODE_309_length_3713_cov_12.835976</strong></dd>

<dd><strong>NODE_335_length_3377_cov_11.732992</strong></dd>

<dd><strong>NODE_337_length_3365_cov_11.337160</strong></dd>

<dd><strong>NODE_352_length_3244_cov_15.563813</strong></dd>

<dd><strong>NODE_353_length_3238_cov_11.065975</strong></dd>

<dd><strong>NODE_376_length_3047_cov_17.076203</strong></dd>

<dd><strong>NODE_384_length_2985_cov_15.617747</strong></dd>

<dd><strong>NODE_386_length_2947_cov_16.119295</strong></dd>

<dd><strong>NODE_436_length_2650_cov_24.480925</strong></dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>








<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_7_length_101522_cov_12.966659 - Region 1 - arylpolyene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-17"></div> <span class="help-tooltip" id="region_svg-help-17">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_7_length_101522_cov_12.966659.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="NODE_7_length_101522_cov_12.966659_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,460 - 47,655 nt. (total: 42,196 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">arylpolyene: (APE_KS1 or APE_KS2)</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-18"></div> <span class="help-tooltip" id="focus-panel-help-18">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-nrps_pks" data-name="r2c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c1-tigrfam-details" data-name="r2c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-7"></div> <span class="help-tooltip" id="gt-help-tooltip-help-7">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-details-svg" data-filename="NODE_7_length_101522_cov_12.966659_r2c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">arylpolyene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus sp. CS93</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184: 0-7298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514: 0-37469">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.48, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Other</td>

<td>aristeromycin</td>


<td>Streptomyces citricolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>agrobactin</td>


<td>Agrobacterium tumefaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882: 0-43357">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000243: 338-14804">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000243/index.html#r1c1" target="_new">BGC0000243</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>nonactin, monactin, dinactin, trinactin, tetranactin</td>


<td>Streptomyces griseus subsp. griseus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197: 0-12679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>frankiamicin</td>


<td>Frankia sp. EAN1pec</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001987: 0-14689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001987/index.html#r1c1" target="_new">BGC0001987</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>koreenceine A, koreenceine B, koreenceine C, koreenceine D</td>


<td>Pseudomonas koreensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001651: 0-10711">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001651/index.html#r1c1" target="_new">BGC0001651</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.22</td>
<td>Other</td>

<td>legionaminic acid</td>


<td>Tannerella forsythia 92A2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002069: 0-10624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002069/index.html#r1c1" target="_new">BGC0002069</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Other (Aminocoumarin)</td>

<td>styrolide A, styrolide B</td>


<td>Pseudomonas fluorescens</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>bacilysin</td>
<td>Bacillus sp. CS93</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184: 0-7298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>bacilysin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514: 0-37469">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.48, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>aristeromycin</td>
<td>Streptomyces citricolor</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>agrobactin</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882: 0-43357">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000243: 338-14804">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000243/index.html#r1c1" target="_new">BGC0000243</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>nonactin, monactin, dinactin, trinactin, tetranactin</td>
<td>Streptomyces griseus subsp. griseus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197: 0-12679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>frankiamicin</td>
<td>Frankia sp. EAN1pec</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001987: 0-14689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001987/index.html#r1c1" target="_new">BGC0001987</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>koreenceine A, koreenceine B, koreenceine C, koreenceine D</td>
<td>Pseudomonas koreensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001651: 0-10711">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001651/index.html#r1c1" target="_new">BGC0001651</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>legionaminic acid</td>
<td>Tannerella forsythia 92A2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002069: 0-10624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002069/index.html#r1c1" target="_new">BGC0002069</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other (Aminocoumarin)</td>
<td>styrolide A, styrolide B</td>
<td>Pseudomonas fluorescens</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NC_004116 (635692-677888): Streptococcus agalactiae 2603V/R, complete sequence (87% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NC_007432 (715194-757390): Streptococcus agalactiae A909, complete sequence (87% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NZ_CP007631 (703097-745305): Streptococcus agalactiae strain NGBS061 chromoso... (87% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NZ_LR134265 (1758766-1800962): Streptococcus agalactiae strain NCTC8184 chrom... (89% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NZ_CP013908 (673780-715976): Streptococcus agalactiae strain GBS-M002 chromos... (87% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NZ_HG939456 (598483-640679): Streptococcus agalactiae COH1 chromosome I, comp... (89% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NZ_CQBY01000001 (20027-62223): Streptococcus agalactiae strain DE-NI-010, who... (82% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_8.svg">
NZ_CP007632 (648807-691003): Streptococcus agalactiae strain NGBS572 chromoso... (82% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_9.svg">
NZ_AP020310 (666345-708541): Streptococcus agalactiae strain GCMC97051 chromo... (83% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_10.svg">
NZ_LT545678 (684363-726559): Streptococcus agalactiae isolate SA111 chromosom... (82% of genes show similarity), arylpolyene
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-8"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-8">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="NODE_7_length_101522_cov_12.966659_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-9"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-9">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-tigrfam-details-svg" data-filename="NODE_7_length_101522_cov_12.966659_r2c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c1-nrps_pks_monomers" data-name="r2c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r2c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-6"></div> <span class="help-tooltip" id="nrps-monomers-help-6">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg2_33</span></strong>: pk
<div class="collapser collapser-target-ctg2_33 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>PKS_AT (85..218)</strong>: pk
<div class="collapser collapser-target-ctg2_33 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  (unknown)
<div class="collapser collapser-target-ctg2_33-ATSignature collapser-level-none"> </div><div class="collapser-content">
No matches above 50%
</div><br>

minowa:  trans-1,2-CPDA
<div class="collapser collapser-target-ctg2_33-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>Propionyl-CoA: 0.0</dt>

<dd></dd><dt>Methylmalonyl-CoA: 0.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 0.0</dt>

<dd></dd><dt>Malonyl-CoA: 0.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Benzoyl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>





<div class="page" id='r47c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_125_length_11688_cov_13.230809 - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-19"></div> <span class="help-tooltip" id="region_svg-help-19">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_125_length_11688_cov_13.230809.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r47c1-svg" data-filename="NODE_125_length_11688_cov_13.230809_r47c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 11,688 nt. (total: 11,688 nt)

<a class="cluster-rules-header" id="r47c1-rules-header" href="#r47c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r47c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>

<div class="region-svg-container">
<div id='r47c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-20"></div> <span class="help-tooltip" id="focus-panel-help-20">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r47c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r47c1-gene-table-details" data-name="r47c1-gene-table-details">
Gene overview
</div>





<div class="body-details-header r47c1-MIBiG-cluster-compare" data-name="r47c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r47c1-clusterblast" data-name="r47c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r47c1-knownclusterblast" data-name="r47c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r47c1-subclusterblast" data-name="r47c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r47c1-pfam-details" data-name="r47c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r47c1-tigrfam-details" data-name="r47c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r47c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-14"></div> <span class="help-tooltip" id="gt-help-tooltip-help-14">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r47c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r47c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r47c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r47c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>





<div class="body-details-section r47c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-10"></div> <span class="help-tooltip" id="MIBiG-help-10">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r47c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r47c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r47c1"></div>
<div id="comparison-MIBiG-r47c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T3PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002561: 0-17947">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002561/index.html#r1c1" target="_new">BGC0002561</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Alkaloid</td>

<td>phenazine SA, phenazine SB, phenazine SC</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.46, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>viguiepinol</td>


<td>Streptomyces sp. KO-3988</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000205: 1474-9885">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000205/index.html#r1c1" target="_new">BGC0000205</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>bryostatin</td>


<td>Candidatus Endobugula sertula</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000948: 0-2463">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000948/index.html#r1c1" target="_new">BGC0000948</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>α-galactosylceramide</td>


<td>Bacteroides fragilis NCTC 9343</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000016: 226-5727">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000016/index.html#r1c1" target="_new">BGC0000016</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Other</td>

<td>amphotericin B</td>


<td>Streptomyces nodosus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000934: 146-25006">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000934/index.html#r1c1" target="_new">BGC0000934</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.47, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Other (Phenazine)</td>

<td>5-acetyl-5,10-dihydrophenazine-1-carboxylic acid, 5-(2-hydroxyacetyl)-5,10-dihydrophenazine-1-carboxylic acid, endophenazine A1, endophenazine F, endophenazine G</td>


<td>Kitasatospora sp. HKI 714</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001080: 0-17804">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001080/index.html#r1c1" target="_new">BGC0001080</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.46, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.10</td>
<td>Other (Phenazine)</td>

<td>endophenazine A, endophenazine B</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002594: 653-30873">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002594/index.html#r1c1" target="_new">BGC0002594</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>orcinol, orsellinic acid</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000872: 0-35385">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000872/index.html#r1c1" target="_new">BGC0000872</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Other</td>

<td>A-90289 A, A-90289 B</td>


<td>Streptomyces sp. SANK 60405</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001076: 0-29709">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001076/index.html#r1c1" target="_new">BGC0001076</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.46, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Other (Nucleoside)</td>

<td>liposidomycin B, liposidomycin A, liposidomycin C, liposidomycin Z, liposidomycin H, liposidomycin G, liposidomycin L, liposidomycin M, liposidomycin K, liposidomycin N</td>


<td>Streptomyces sp. SN-1061M</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r47c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002561: 0-17947">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002561/index.html#r1c1" target="_new">BGC0002561</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Alkaloid</td>
<td>phenazine SA, phenazine SB, phenazine SC</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.46, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>viguiepinol</td>
<td>Streptomyces sp. KO-3988</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000205: 1474-9885">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000205/index.html#r1c1" target="_new">BGC0000205</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Polyketide</td>
<td>bryostatin</td>
<td>Candidatus Endobugula sertula</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000948: 0-2463">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000948/index.html#r1c1" target="_new">BGC0000948</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>α-galactosylceramide</td>
<td>Bacteroides fragilis NCTC 9343</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000016: 226-5727">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000016/index.html#r1c1" target="_new">BGC0000016</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Other</td>
<td>amphotericin B</td>
<td>Streptomyces nodosus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000934: 146-25006">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000934/index.html#r1c1" target="_new">BGC0000934</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.47, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other (Phenazine)</td>
<td>5-acetyl-5,10-dihydrophenazine-1-carboxylic acid, 5-(2-hydroxyacetyl)-5,10-dihydrophenazine-1-carboxylic acid, endophenazine A1, endophenazine F, endophenazine G</td>
<td>Kitasatospora sp. HKI 714</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001080: 0-17804">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001080/index.html#r1c1" target="_new">BGC0001080</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.46, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other (Phenazine)</td>
<td>endophenazine A, endophenazine B</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002594: 653-30873">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002594/index.html#r1c1" target="_new">BGC0002594</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Polyketide</td>
<td>orcinol, orsellinic acid</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000872: 0-35385">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000872/index.html#r1c1" target="_new">BGC0000872</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Other</td>
<td>A-90289 A, A-90289 B</td>
<td>Streptomyces sp. SANK 60405</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001076: 0-29709">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001076/index.html#r1c1" target="_new">BGC0001076</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.46, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Other (Nucleoside)</td>
<td>liposidomycin B, liposidomycin A, liposidomycin C, liposidomycin Z, liposidomycin H, liposidomycin G, liposidomycin L, liposidomycin M, liposidomycin K, liposidomycin N</td>
<td>Streptomyces sp. SN-1061M</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r47c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-11"></div> <span class="help-tooltip" id="cb-general-help-11">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r47c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r47c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r47c1_1.svg">
NZ_LR134265 (1031357-1072635): Streptococcus agalactiae strain NCTC8184 chrom... (34% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r47c1_2.svg">
NZ_CP007572 (1349890-1391168): Streptococcus agalactiae strain GBS6 chromosom... (37% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r47c1_3.svg">
NC_004116 (1302940-1344218): Streptococcus agalactiae 2603V/R, complete sequence (32% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r47c1_4.svg">
NZ_CP006910 (1237677-1278955): Streptococcus agalactiae CNCTC 10/84 chromosom... (36% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r47c1_5.svg">
NZ_CP007632 (1224388-1265666): Streptococcus agalactiae strain NGBS572 chromo... (36% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r47c1_6.svg">
NZ_CP010319 (1236025-1277303): Streptococcus agalactiae strain GBS85147 chrom... (32% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r47c1_7.svg">
NZ_ALSD01000048 (23951-65229): Streptococcus agalactiae MRI Z1-213 ctg7180000... (32% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r47c1_8.svg">
NZ_JAIMFG010000003 (45865-87089): Streptococcus agalactiae strain PZD2 PZD2 S... (35% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r47c1_9.svg">
NZ_LT545678 (1423738-1465016): Streptococcus agalactiae isolate SA111 chromos... (31% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r47c1_10.svg">
NZ_GL636070 (893505-934783): Streptococcus agalactiae ATCC 13813 SCAFFOLD1, w... (36% of genes show similarity), T3PKS
</option>

</select>
<button id= "clusterblast-r47c1-download">Download graphic</button>
</div>
<div id="clusterblast-r47c1-svg"></div>

</div>
</div>

<div class="body-details-section r47c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-12"></div> <span class="help-tooltip" id="cb-known-help-12">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r47c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-13"></div> <span class="help-tooltip" id="cb-sub-help-13">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r47c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-15"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-15">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r47c1-pfam-details-svg" data-filename="NODE_125_length_11688_cov_13.230809_r47c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r47c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r47c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-16"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-16">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r47c1-tigrfam-details-svg" data-filename="NODE_125_length_11688_cov_13.230809_r47c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r47c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">













</div>













</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>