
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>MG_323757.bin1 - 2 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="MG_323757.bin1 - 2 record(s) - 2 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/MG_323757.bin1/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.1.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="MG_323757.bin1.zip">Download all results</a></li>

<li><a href="MG_323757.bin1.gbk">Download GenBank summary file</a></li>
<li><a href="MG_323757.bin1.json">Download JSON results file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton RiPP RiPP-like r59c1">
<a href="#r59c1">59.1</a>
</div>



<div class="regbutton RiPP RiPP-like r82c1">
<a href="#r82c1">82.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NODE_83_length_8923_cov_7.096640</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-59" data-filename="NODE_83_length_8923_cov_7.096640_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-59" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r59c1">
<td class="regbutton RiPP RiPP-like r59c1">
<a href="#r59c1">Region&nbsp59.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">5,996</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_110_length_7016_cov_6.405832</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-82" data-filename="NODE_110_length_7016_cov_6.405832_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-82" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r82c1">
<td class="regbutton RiPP RiPP-like r82c1">
<a href="#r82c1">Region&nbsp82.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">7,016</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r59c1">
<td class="regbutton RiPP RiPP-like r59c1">
<a href="#r59c1">Region&nbsp59.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">5,996</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r82c1">
<td class="regbutton RiPP RiPP-like r82c1">
<a href="#r82c1">Region&nbsp82.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">7,016</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NODE_12_length_60766_cov_7.504011</strong></dd>

<dd><strong>NODE_14_length_56370_cov_7.332469</strong></dd>

<dd><strong>NODE_16_length_45026_cov_7.459985</strong></dd>

<dd><strong>NODE_18_length_43053_cov_7.059398</strong></dd>

<dd><strong>NODE_23_length_32447_cov_7.189183</strong></dd>

<dd><strong>NODE_24_length_28023_cov_6.762300</strong></dd>

<dd><strong>NODE_25_length_27592_cov_5.900207</strong></dd>

<dd><strong>NODE_26_length_26293_cov_6.518408</strong></dd>

<dd><strong>NODE_28_length_26092_cov_6.799939</strong></dd>

<dd><strong>NODE_29_length_25733_cov_8.241179</strong></dd>

<dd><strong>NODE_30_length_25471_cov_6.894555</strong></dd>

<dd><strong>NODE_31_length_23122_cov_7.547969</strong></dd>

<dd><strong>NODE_32_length_23100_cov_7.532480</strong></dd>

<dd><strong>NODE_33_length_21769_cov_7.884729</strong></dd>

<dd><strong>NODE_34_length_21065_cov_7.635459</strong></dd>

<dd><strong>NODE_35_length_19753_cov_5.791248</strong></dd>

<dd><strong>NODE_37_length_18376_cov_8.800175</strong></dd>

<dd><strong>NODE_38_length_17988_cov_6.939720</strong></dd>

<dd><strong>NODE_39_length_17929_cov_6.124874</strong></dd>

<dd><strong>NODE_40_length_17728_cov_7.740338</strong></dd>

<dd><strong>NODE_43_length_17033_cov_5.880905</strong></dd>

<dd><strong>NODE_44_length_17013_cov_7.484196</strong></dd>

<dd><strong>NODE_45_length_15812_cov_6.176303</strong></dd>

<dd><strong>NODE_46_length_15484_cov_6.322185</strong></dd>

<dd><strong>NODE_47_length_15211_cov_7.820665</strong></dd>

<dd><strong>NODE_48_length_14798_cov_6.037509</strong></dd>

<dd><strong>NODE_49_length_14159_cov_7.528148</strong></dd>

<dd><strong>NODE_50_length_13892_cov_5.715184</strong></dd>

<dd><strong>NODE_51_length_13723_cov_6.660228</strong></dd>

<dd><strong>NODE_52_length_13241_cov_7.721978</strong></dd>

<dd><strong>NODE_53_length_12972_cov_5.663312</strong></dd>

<dd><strong>NODE_54_length_12905_cov_8.109961</strong></dd>

<dd><strong>NODE_55_length_12480_cov_5.396137</strong></dd>

<dd><strong>NODE_56_length_12479_cov_7.201626</strong></dd>

<dd><strong>NODE_58_length_12408_cov_7.390593</strong></dd>

<dd><strong>NODE_59_length_12364_cov_5.573239</strong></dd>

<dd><strong>NODE_60_length_11415_cov_5.782658</strong></dd>

<dd><strong>NODE_61_length_11316_cov_6.212415</strong></dd>

<dd><strong>NODE_62_length_11264_cov_4.990454</strong></dd>

<dd><strong>NODE_64_length_11123_cov_6.025569</strong></dd>

<dd><strong>NODE_65_length_11084_cov_5.813945</strong></dd>

<dd><strong>NODE_66_length_10920_cov_6.771468</strong></dd>

<dd><strong>NODE_67_length_10914_cov_7.681923</strong></dd>

<dd><strong>NODE_68_length_10701_cov_6.182510</strong></dd>

<dd><strong>NODE_69_length_10663_cov_6.080882</strong></dd>

<dd><strong>NODE_70_length_10634_cov_6.024955</strong></dd>

<dd><strong>NODE_71_length_10607_cov_5.992039</strong></dd>

<dd><strong>NODE_72_length_10225_cov_6.108063</strong></dd>

<dd><strong>NODE_73_length_10039_cov_5.668570</strong></dd>

<dd><strong>NODE_74_length_9928_cov_4.598197</strong></dd>

<dd><strong>NODE_75_length_9871_cov_5.263957</strong></dd>

<dd><strong>NODE_76_length_9799_cov_5.569889</strong></dd>

<dd><strong>NODE_77_length_9780_cov_5.421594</strong></dd>

<dd><strong>NODE_78_length_9660_cov_5.394066</strong></dd>

<dd><strong>NODE_79_length_9312_cov_5.103813</strong></dd>

<dd><strong>NODE_80_length_9276_cov_5.755883</strong></dd>

<dd><strong>NODE_81_length_8997_cov_6.219638</strong></dd>

<dd><strong>NODE_82_length_8932_cov_6.465811</strong></dd>

<dd><strong>NODE_84_length_8902_cov_5.751441</strong></dd>

<dd><strong>NODE_85_length_8821_cov_5.297399</strong></dd>

<dd><strong>NODE_86_length_8690_cov_5.295889</strong></dd>

<dd><strong>NODE_87_length_8595_cov_6.311475</strong></dd>

<dd><strong>NODE_88_length_8580_cov_6.460762</strong></dd>

<dd><strong>NODE_89_length_8572_cov_5.961958</strong></dd>

<dd><strong>NODE_90_length_8393_cov_6.165148</strong></dd>

<dd><strong>NODE_92_length_8193_cov_7.830671</strong></dd>

<dd><strong>NODE_93_length_8037_cov_6.179529</strong></dd>

<dd><strong>NODE_94_length_7938_cov_6.625650</strong></dd>

<dd><strong>NODE_95_length_7887_cov_5.073672</strong></dd>

<dd><strong>NODE_96_length_7836_cov_6.112196</strong></dd>

<dd><strong>NODE_97_length_7832_cov_6.243410</strong></dd>

<dd><strong>NODE_98_length_7814_cov_7.616059</strong></dd>

<dd><strong>NODE_101_length_7470_cov_5.304383</strong></dd>

<dd><strong>NODE_102_length_7419_cov_6.651276</strong></dd>

<dd><strong>NODE_103_length_7296_cov_5.546471</strong></dd>

<dd><strong>NODE_104_length_7292_cov_6.539174</strong></dd>

<dd><strong>NODE_105_length_7238_cov_5.841710</strong></dd>

<dd><strong>NODE_106_length_7216_cov_5.717218</strong></dd>

<dd><strong>NODE_108_length_7106_cov_5.040987</strong></dd>

<dd><strong>NODE_109_length_7068_cov_5.236988</strong></dd>

<dd><strong>NODE_112_length_6841_cov_5.858974</strong></dd>

<dd><strong>NODE_113_length_6659_cov_5.371593</strong></dd>

<dd><strong>NODE_114_length_6615_cov_6.135213</strong></dd>

<dd><strong>NODE_115_length_6592_cov_6.819030</strong></dd>

<dd><strong>NODE_117_length_6463_cov_5.220037</strong></dd>

<dd><strong>NODE_118_length_6348_cov_6.943747</strong></dd>

<dd><strong>NODE_119_length_6345_cov_5.317965</strong></dd>

<dd><strong>NODE_121_length_6247_cov_5.687339</strong></dd>

<dd><strong>NODE_122_length_6171_cov_5.021910</strong></dd>

<dd><strong>NODE_123_length_6131_cov_4.746050</strong></dd>

<dd><strong>NODE_125_length_6015_cov_5.979530</strong></dd>

<dd><strong>NODE_126_length_5912_cov_5.446304</strong></dd>

<dd><strong>NODE_127_length_5846_cov_6.214471</strong></dd>

<dd><strong>NODE_128_length_5831_cov_6.298130</strong></dd>

<dd><strong>NODE_129_length_5673_cov_5.038982</strong></dd>

<dd><strong>NODE_133_length_5423_cov_4.805142</strong></dd>

<dd><strong>NODE_134_length_5011_cov_5.656780</strong></dd>

<dd><strong>NODE_135_length_4917_cov_6.372275</strong></dd>

<dd><strong>NODE_136_length_4884_cov_5.667840</strong></dd>

<dd><strong>NODE_137_length_4595_cov_6.616740</strong></dd>

<dd><strong>NODE_138_length_4595_cov_5.241630</strong></dd>

<dd><strong>NODE_139_length_4586_cov_7.102626</strong></dd>

<dd><strong>NODE_140_length_4546_cov_5.182365</strong></dd>

<dd><strong>NODE_141_length_4543_cov_5.272282</strong></dd>

<dd><strong>NODE_142_length_4515_cov_6.082287</strong></dd>

<dd><strong>NODE_143_length_4476_cov_6.140014</strong></dd>

<dd><strong>NODE_144_length_4468_cov_5.152277</strong></dd>

<dd><strong>NODE_145_length_4446_cov_5.092462</strong></dd>

<dd><strong>NODE_147_length_4314_cov_5.310636</strong></dd>

<dd><strong>NODE_149_length_4184_cov_4.964640</strong></dd>

<dd><strong>NODE_150_length_4156_cov_6.046330</strong></dd>

<dd><strong>NODE_151_length_4104_cov_5.500617</strong></dd>

<dd><strong>NODE_152_length_4074_cov_5.430953</strong></dd>

<dd><strong>NODE_153_length_4072_cov_5.189445</strong></dd>

<dd><strong>NODE_154_length_4021_cov_4.516641</strong></dd>

<dd><strong>NODE_155_length_4020_cov_4.704666</strong></dd>

<dd><strong>NODE_156_length_3912_cov_5.723619</strong></dd>

<dd><strong>NODE_157_length_3895_cov_4.640104</strong></dd>

<dd><strong>NODE_158_length_3778_cov_6.083535</strong></dd>

<dd><strong>NODE_159_length_3758_cov_5.452876</strong></dd>

<dd><strong>NODE_160_length_3667_cov_7.065615</strong></dd>

<dd><strong>NODE_161_length_3623_cov_7.905269</strong></dd>

<dd><strong>NODE_162_length_3452_cov_5.972623</strong></dd>

<dd><strong>NODE_163_length_3442_cov_4.932979</strong></dd>

<dd><strong>NODE_164_length_3324_cov_4.513919</strong></dd>

<dd><strong>NODE_165_length_3323_cov_6.945838</strong></dd>

<dd><strong>NODE_166_length_3248_cov_5.565925</strong></dd>

<dd><strong>NODE_168_length_3216_cov_5.355267</strong></dd>

<dd><strong>NODE_169_length_3202_cov_5.831586</strong></dd>

<dd><strong>NODE_170_length_3202_cov_4.527486</strong></dd>

<dd><strong>NODE_171_length_3152_cov_6.467226</strong></dd>

<dd><strong>NODE_172_length_3146_cov_7.153025</strong></dd>

<dd><strong>NODE_173_length_3139_cov_5.880026</strong></dd>

<dd><strong>NODE_175_length_3054_cov_5.030677</strong></dd>

<dd><strong>NODE_176_length_3038_cov_7.307744</strong></dd>

<dd><strong>NODE_177_length_3024_cov_6.623442</strong></dd>

<dd><strong>NODE_180_length_2880_cov_5.054159</strong></dd>

<dd><strong>NODE_184_length_2752_cov_4.459029</strong></dd>

<dd><strong>NODE_185_length_2679_cov_5.764101</strong></dd>

<dd><strong>NODE_186_length_2654_cov_5.128126</strong></dd>

<dd><strong>NODE_189_length_2615_cov_7.715234</strong></dd>

<dd><strong>NODE_190_length_2579_cov_6.607765</strong></dd>

<dd><strong>NODE_191_length_2570_cov_4.735984</strong></dd>

<dd><strong>NODE_193_length_2530_cov_6.785859</strong></dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>








<div class="page" id='r59c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_83_length_8923_cov_7.096640 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-15"></div> <span class="help-tooltip" id="region_svg-help-15">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_83_length_8923_cov_7.096640.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r59c1-svg" data-filename="NODE_83_length_8923_cov_7.096640_r59c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 5,996 nt. (total: 5,996 nt)

<a class="cluster-rules-header" id="r59c1-rules-header" href="#r59c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r59c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193 or (YcaO or TIGR03604))</div>

<div class="region-svg-container">
<div id='r59c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-16"></div> <span class="help-tooltip" id="focus-panel-help-16">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r59c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r59c1-gene-table-details" data-name="r59c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r59c1-MIBiG-cluster-compare" data-name="r59c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r59c1-clusterblast" data-name="r59c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r59c1-knownclusterblast" data-name="r59c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r59c1-subclusterblast" data-name="r59c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r59c1-pfam-details" data-name="r59c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r59c1-tigrfam-details" data-name="r59c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r59c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-5"></div> <span class="help-tooltip" id="gt-help-tooltip-help-5">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r59c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r59c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r59c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r59c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r59c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<div><p>No matches found.</p></div>

</div>
</div>



<div class="body-details-section r59c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r59c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r59c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r59c1_1.svg">
NZ_NGLI01000002 (555810-572363): Enterococcus sp. 3G1 DIV0629 scaffold00002, ... (11% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r59c1_2.svg">
NZ_CP078505 (3139533-3154876): Enterococcus gallinarum strain fecal iso1 chro... (13% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r59c1_3.svg">
NZ_KZ846093 (59374-74717): Enterococcus faecium strain 131EA1 aekeh-supercont... (14% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r59c1_4.svg">
NZ_KZ846230 (55734-72409): Enterococcus faecium strain 235EA1 aekdA-supercont... (13% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r59c1_5.svg">
NZ_KB946108 (78814-95380): Enterococcus faecium EnGen0263 strain UAA1280 acvJ... (10% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r59c1_6.svg">
NZ_CP072983 (327303-342619): Lactobacillus intestinalis strain DSM 6629 chrom... (17% of genes show similarity), RiPP-like
</option>

</select>
<button id= "clusterblast-r59c1-download">Download graphic</button>
</div>
<div id="clusterblast-r59c1-svg"></div>

</div>
</div>

<div class="body-details-section r59c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r59c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r59c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-6"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-6">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r59c1-pfam-details-svg" data-filename="NODE_83_length_8923_cov_7.096640_r59c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r59c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r59c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-7"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-7">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r59c1-tigrfam-details-svg" data-filename="NODE_83_length_8923_cov_7.096640_r59c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r59c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r82c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_110_length_7016_cov_6.405832 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-17"></div> <span class="help-tooltip" id="region_svg-help-17">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_110_length_7016_cov_6.405832.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r82c1-svg" data-filename="NODE_110_length_7016_cov_6.405832_r82c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 7,016 nt. (total: 7,016 nt)

<a class="cluster-rules-header" id="r82c1-rules-header" href="#r82c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r82c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193 or (YcaO or TIGR03604))</div>

<div class="region-svg-container">
<div id='r82c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-18"></div> <span class="help-tooltip" id="focus-panel-help-18">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r82c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r82c1-gene-table-details" data-name="r82c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r82c1-MIBiG-cluster-compare" data-name="r82c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r82c1-clusterblast" data-name="r82c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r82c1-knownclusterblast" data-name="r82c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r82c1-subclusterblast" data-name="r82c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r82c1-pfam-details" data-name="r82c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r82c1-tigrfam-details" data-name="r82c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r82c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-12"></div> <span class="help-tooltip" id="gt-help-tooltip-help-12">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r82c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r82c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r82c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r82c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r82c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-8"></div> <span class="help-tooltip" id="MIBiG-help-8">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r82c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r82c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r82c1"></div>
<div id="comparison-MIBiG-r82c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002585: 998-10750">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002585/index.html#r1c1" target="_new">BGC0002585</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.48, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>ubericin K</td>


<td>Streptococcus uberis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000617: 1088-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000617/index.html#r1c1" target="_new">BGC0000617</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.48, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>RiPP</td>

<td>coagulin</td>


<td>Bacillus coagulans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002579: 0-7721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002579/index.html#r1c1" target="_new">BGC0002579</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.47, order:0.46, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>RiPP</td>

<td>carnobacteriocin XY</td>


<td>Carnobacterium maltaromaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000558: 856-5167">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000558/index.html#r1c1" target="_new">BGC0000558</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>sublancin 168</td>


<td>Bacillus subtilis subsp. subtilis str. 168</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002577: 535-4868">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002577/index.html#r1c1" target="_new">BGC0002577</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.50, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>lactococcin G</td>


<td>Lactococcus lactis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001388: 140-6188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001388/index.html#r1c1" target="_new">BGC0001388</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.45, order:0.46, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>gassericin E</td>


<td>Lactobacillus gasseri</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000619: 155-6278">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000619/index.html#r1c1" target="_new">BGC0000619</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.45, order:0.46, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>gassericin T</td>


<td>Lactobacillus gasseri</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001602: 155-6278">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001602/index.html#r1c1" target="_new">BGC0001602</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.45, order:0.46, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>gassericin-T</td>


<td>Lactobacillus gasseri</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000624: 46-7740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000624/index.html#r1c1" target="_new">BGC0000624</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.47, order:0.45, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>salivaricin CRL1328 α peptide, salivaricin CRL1328 β peptide</td>


<td>Lactobacillus salivarius</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001863: 699-11883">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001863/index.html#r1c1" target="_new">BGC0001863</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP, Terpene</td>

<td>bacillicn CER074</td>


<td>Bacillus mycoides</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r82c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002585: 998-10750">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002585/index.html#r1c1" target="_new">BGC0002585</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.48, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>ubericin K</td>
<td>Streptococcus uberis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000617: 1088-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000617/index.html#r1c1" target="_new">BGC0000617</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.48, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>RiPP</td>
<td>coagulin</td>
<td>Bacillus coagulans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002579: 0-7721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002579/index.html#r1c1" target="_new">BGC0002579</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.47, order:0.46, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>RiPP</td>
<td>carnobacteriocin XY</td>
<td>Carnobacterium maltaromaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000558: 856-5167">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000558/index.html#r1c1" target="_new">BGC0000558</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>sublancin 168</td>
<td>Bacillus subtilis subsp. subtilis str. 168</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002577: 535-4868">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002577/index.html#r1c1" target="_new">BGC0002577</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.50, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>lactococcin G</td>
<td>Lactococcus lactis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001388: 140-6188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001388/index.html#r1c1" target="_new">BGC0001388</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.45, order:0.46, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>gassericin E</td>
<td>Lactobacillus gasseri</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000619: 155-6278">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000619/index.html#r1c1" target="_new">BGC0000619</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.45, order:0.46, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>gassericin T</td>
<td>Lactobacillus gasseri</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001602: 155-6278">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001602/index.html#r1c1" target="_new">BGC0001602</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.45, order:0.46, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>gassericin-T</td>
<td>Lactobacillus gasseri</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000624: 46-7740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000624/index.html#r1c1" target="_new">BGC0000624</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.47, order:0.45, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>RiPP</td>
<td>salivaricin CRL1328 α peptide, salivaricin CRL1328 β peptide</td>
<td>Lactobacillus salivarius</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001863: 699-11883">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001863/index.html#r1c1" target="_new">BGC0001863</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP, Terpene</td>
<td>bacillicn CER074</td>
<td>Bacillus mycoides</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r82c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-9"></div> <span class="help-tooltip" id="cb-general-help-9">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r82c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r82c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r82c1_1.svg">
NZ_KB946108 (78814-95380): Enterococcus faecium EnGen0263 strain UAA1280 acvJ... (15% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r82c1_2.svg">
NZ_NGLI01000002 (555810-572363): Enterococcus sp. 3G1 DIV0629 scaffold00002, ... (16% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r82c1_3.svg">
NZ_CP078505 (3139533-3154876): Enterococcus gallinarum strain fecal iso1 chro... (20% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r82c1_4.svg">
NZ_KZ846093 (59374-74717): Enterococcus faecium strain 131EA1 aekeh-supercont... (21% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r82c1_5.svg">
NZ_KZ846230 (55734-72409): Enterococcus faecium strain 235EA1 aekdA-supercont... (20% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r82c1_6.svg">
NZ_KB946310 (115445-132208): Enterococcus pallens ATCC BAA-351 acvKb-supercon... (18% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r82c1_7.svg">
NZ_SPHN01000003 (68221-80384): Enterococcus sp. BSD2780061688st2 D3 strain BS... (33% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r82c1_8.svg">
NZ_CP116581 (2484979-2497142): Enterococcus durans strain R59-1 chromosome, c... (33% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r82c1_9.svg">
NZ_CP065535 (2360067-2371726): Enterococcus durans strain KCTC 13289 chromoso... (40% of genes show similarity), RiPP-like
</option>

<option value = "svg/clusterblast_r82c1_10.svg">
NZ_LR607372 (1332690-1344853): Enterococcus durans strain 4928STDY7387704 chr... (33% of genes show similarity), RiPP-like
</option>

</select>
<button id= "clusterblast-r82c1-download">Download graphic</button>
</div>
<div id="clusterblast-r82c1-svg"></div>

</div>
</div>

<div class="body-details-section r82c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-10"></div> <span class="help-tooltip" id="cb-known-help-10">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r82c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-11"></div> <span class="help-tooltip" id="cb-sub-help-11">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r82c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-13"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-13">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r82c1-pfam-details-svg" data-filename="NODE_110_length_7016_cov_6.405832_r82c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r82c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r82c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-14"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-14">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r82c1-tigrfam-details-svg" data-filename="NODE_110_length_7016_cov_6.405832_r82c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r82c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>