
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>MG_323757.bin3 - 1 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="MG_323757.bin3 - 1 record(s) - 1 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/Lactobacillus_iners1/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.1.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="MG_323757.bin3.zip">Download all results</a></li>
        
          <li><a href="MG_323757.bin3.gbk">Download GenBank summary file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            <div class="regbutton RiPP lanthipeptide-class-ii r15c1">
              <a href="#r15c1">15.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>c00015_NODE_63..</strong> (original name was: NODE_63_length_11229_cov_186.926347)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-15" data-filename="c00015_NODE_63.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-15" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r15c1">
     <td class="regbutton RiPP lanthipeptide-class-ii r15c1">
      <a href="#r15c1">Region&nbsp15.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-ii" target="_blank">lanthipeptide-class-ii</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,229</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000532/1" target="_blank">mutacin II</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r15c1">
     <td class="regbutton RiPP lanthipeptide-class-ii r15c1">
      <a href="#r15c1">Region&nbsp15.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-ii" target="_blank">lanthipeptide-class-ii</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,229</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000532/1" target="_blank">mutacin II</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>c00001_NODE_3_..</strong> (original name was: NODE_3_length_326906_cov_141.422088)</dd>
       
         <dd><strong>c00002_NODE_5_..</strong> (original name was: NODE_5_length_209495_cov_170.061163)</dd>
       
         <dd><strong>c00003_NODE_7_..</strong> (original name was: NODE_7_length_109574_cov_201.170774)</dd>
       
         <dd><strong>c00004_NODE_8_..</strong> (original name was: NODE_8_length_107775_cov_146.249443)</dd>
       
         <dd><strong>c00005_NODE_9_..</strong> (original name was: NODE_9_length_92848_cov_188.148438)</dd>
       
         <dd><strong>c00006_NODE_11..</strong> (original name was: NODE_11_length_75853_cov_189.540450)</dd>
       
         <dd><strong>c00007_NODE_13..</strong> (original name was: NODE_13_length_58994_cov_162.688933)</dd>
       
         <dd><strong>c00008_NODE_15..</strong> (original name was: NODE_15_length_45166_cov_157.736339)</dd>
       
         <dd><strong>c00009_NODE_17..</strong> (original name was: NODE_17_length_43236_cov_178.105671)</dd>
       
         <dd><strong>c00010_NODE_19..</strong> (original name was: NODE_19_length_40684_cov_239.359817)</dd>
       
         <dd><strong>c00011_NODE_21..</strong> (original name was: NODE_21_length_36247_cov_192.252045)</dd>
       
         <dd><strong>c00012_NODE_22..</strong> (original name was: NODE_22_length_36138_cov_135.161821)</dd>
       
         <dd><strong>c00013_NODE_27..</strong> (original name was: NODE_27_length_26189_cov_162.615023)</dd>
       
         <dd><strong>c00014_NODE_57..</strong> (original name was: NODE_57_length_12466_cov_184.642736)</dd>
       
         <dd><strong>c00016_NODE_91..</strong> (original name was: NODE_91_length_8358_cov_149.084668)</dd>
       
         <dd><strong>c00017_NODE_12..</strong> (original name was: NODE_120_length_6330_cov_180.253386)</dd>
       
         <dd><strong>c00018_NODE_13..</strong> (original name was: NODE_130_length_5511_cov_158.555902)</dd>
       
         <dd><strong>c00019_NODE_13..</strong> (original name was: NODE_131_length_5494_cov_163.056628)</dd>
       
         <dd><strong>c00020_NODE_20..</strong> (original name was: NODE_205_length_2276_cov_184.268798)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r15c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00015_NODE_63.. - Region 1 - lanthipeptide-class-ii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-6"></div> <span class="help-tooltip" id="region_svg-help-6">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00015_NODE_63...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r15c1-svg" data-filename="c00015_NODE_63.._r15c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 11,229 nt. (total: 11,229 nt)
        
        <a class="cluster-rules-header" id="r15c1-rules-header" href="#r15c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r15c1-rules">lanthipeptide-class-ii: cds(LANC_like and DUF4135)</div>
      
      <div class="region-svg-container">
       <div id='r15c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-7"></div> <span class="help-tooltip" id="focus-panel-help-7">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r15c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r15c1-clusterblast" data-name="r15c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r15c1-knownclusterblast" data-name="r15c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r15c1-subclusterblast" data-name="r15c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r15c1-lanthipeptides" data-name="r15c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r15c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-1"></div> <span class="help-tooltip" id="cb-general-help-1">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r15c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r15c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r15c1_1.svg">
        NZ_CP049228 (1324304-1347013): Lactobacillus iners strain C0210C1 chromosome,... (36% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r15c1_2.svg">
        NZ_CP044492 (98905-121611): Streptococcus mutans strain T8 chromosome, comple... (33% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r15c1_3.svg">
        NZ_CP037940 (1611546-1634315): Weissella cryptocerci strain 26KH-42 chromosom... (33% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r15c1_4.svg">
        NZ_FBTC01000016 (75047-97834): Leuconostoc inhae strain C120c, whole genome s... (20% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r15c1_5.svg">
        NZ_LN890331 (470021-492808): Leuconostoc gelidum subsp. gasicomitatum KG16-1 ... (22% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r15c1_6.svg">
        NZ_CP023483 (2448685-2471457): Brochothrix thermosphacta strain BI chromosome... (20% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r15c1_7.svg">
        NC_015760 (742809-776926): Streptococcus salivarius CCHSS3, complete genome (14% of genes show similarity), RaS-RiPP,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r15c1_8.svg">
        NZ_GL636070 (1699887-1740767): Streptococcus agalactiae ATCC 13813 SCAFFOLD1,... (10% of genes show similarity), RaS-RiPP,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r15c1_9.svg">
        NZ_CP029398 (1011468-1034249): Streptococcus suis strain HN105 chromosome, co... (19% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r15c1_10.svg">
        NZ_LR738724 (1009251-1032032): Streptococcus suis isolate 9401240 chromosome 1 (20% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r15c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r15c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r15c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-2"></div> <span class="help-tooltip" id="cb-known-help-2">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r15c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r15c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r15c1_1.svg">
        mutacin II
      </option>
      
      <option value = "svg/knownclusterblast_r15c1_2.svg">
        ruminococcin A
      </option>
      
      <option value = "svg/knownclusterblast_r15c1_3.svg">
        lacticin 481
      </option>
      
      <option value = "svg/knownclusterblast_r15c1_4.svg">
        variacin
      </option>
      
    </select>
    <button id="knownclusterblast-r15c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r15c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r15c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-3"></div> <span class="help-tooltip" id="cb-sub-help-3">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r15c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-4"></div> <span class="help-tooltip" id="lanthi-body-help-4">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for ctg15_9 - Putative Class II</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">ctg15_10</span> leader / core peptide
          </dt>
          <dd>
            MEKLNHNAVTSLNEVSDSELDTILGG
            -
            <span class="dha">Dha</span>RWWQGVLP<span class="dhb">Dhb</span>V<span class="dha">Dha</span>HE<span class="cys">C</span>RMN<span class="dha">Dha</span>FQHIF<span class="dhb">Dhb</span><span class="cys">C</span><span class="cys">C</span>
          <dt>
            <span class="serif">allorf_01269_01403</span> leader / core peptide
          </dt>
          <dd>
            LVIDIEDEKQGNIITG
            -
            FKEPEHIQLMILRKL<span class="dhb">Dhb</span>VR<span class="cys">C</span>MRFRWGFII
          <dt>
            <span class="serif">allorf_01361_01540</span> leader / core peptide
          </dt>
          <dd>
            MTQEEYMIQELSHYQFS
            -
            PE<span class="dha">Dha</span>QYLDRKYVRKK<span class="cys">C</span>QNF<span class="dha">Dha</span><span class="dha">Dha</span>ILLMLMVV<span class="dha">Dha</span>WLLILRMRNKEI<span class="dha">Dha</span>
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r15c1-lanthipeptides" data-name="r15c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r15c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-5"></div> <span class="help-tooltip" id="lanthi-side-help-5">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">ctg15_10</span> - Class II</dt>
        <dd>Cleavage pHMM score: 5.10</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 3161.5 Da</dd>
        <dd>Molecular weight: 3163.6 Da</dd>
        <dd>Number of bridges: 3</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 3181.6 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3199.7 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">allorf_01269_01403</span> - Class II</dt>
        <dd>Cleavage pHMM score: -6.80</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 3542.0 Da</dd>
        <dd>Molecular weight: 3544.3 Da</dd>
        <dd>Number of bridges: 1</dd><dt><span class="serif">allorf_01361_01540</span> - Class II</dt>
        <dd>Cleavage pHMM score: -5.50</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 5037.8 Da</dd>
        <dd>Molecular weight: 5041.1 Da</dd>
        <dd>Number of bridges: 1</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 5059.1 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 5077.1 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 5095.1 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 5113.1 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>